%-------------------------------------------------------------------------%
%--------------------Classroom Lecture Model Series-----------------------%
%-------------------------------------------------------------------------%

\begin{document}
\twocolumn
\scriptsize
\begin{frontmatter}
		\title{DRAFT Sample: A Delayed Differential Equation Model of Phytochemicals in GastroIntestinal Cancers}
		\author{\corref{cor1}\fnref{fn1}}
		\cortext[cor1]{Corresponding author}
		\address{The Mathematical Learning Space}
		\ead{http://mathlearningspace.weebly.com}	
\end{frontmatter}	

Introduction:
\begin{enumerate}
\item Objective 1:
\item Objective 2:
\item Objective 3:
\end{enumerate}
Conclusion:

Keywords: Differential Equations, Stability, Phase Space, GastroIntestinal Cancers, Phytochemicals
Vocabulary Words:polyphenols, flavonoids, phytoalexins, phenolic acids, indoles, acylating, alkylating, glycosylating, reactive hydroxy groups methylene groups 

\section{Introduction}



\begin{figure}[H]
\begin{minipage}[b]{0.3\linewidth}
\includegraphics[scale=0.40]{GastroIntestinal_Cancer.png} 
\end{minipage}\hfill
\caption{(a) GastroIntestinal Cancer}
\label{fig:Figure1}
\end{figure} 

\begin{table}[H]\centering
	\begin{tabular}{p{1cm}p{4cm}p{3cm}}
		Article ID & Summary & Comments\\
		\hline
		\hline
	\end{tabular}
\end{table}


\subsection{Gastric Cancer}

Table 1 shows the activity states of the genetic material in Gastric Cancer with (a) overexpression, (b) mutation, (c) reduced expression, (d) amplification and (e) methylation.  These can be grouped in 

\begin{table}[H]
\caption{Gene Analysis for Gastric Cancer}	
\begin{tabular}{p{2cm}p{3cm}p{1cm}}
\hline	
Gene & Category & Reference \\ 
\hline
CDX2 & overexpression & \cite{key400} \\
TERT & overexpression & \cite{key400}  \\
\hline
p53 & mutation &  \cite{key400} \\
APC &  mutation &  \cite{key400} \\
CTNNB1 & mutation &  \cite{key400} \\
KRAS & mutation &  \cite{key400}  \\
NRAS & mutation &  \cite{key400} \\
CDH1 & mutation &  \cite{key400} \\
\hline
MLH1 & methylation & \cite{key400}  \\
\hline
RARB & reduced expression &  \cite{key400}  \\
CDKN1B & reduced expression & \cite{key400} \\
TGFBR1 & reduced expression &  \cite{key400} \\
\hline
ERBB2 & amplification & \cite{key400}  \\
CCNE1 & amplification &  \cite{key400} \\
MET & amplification &  \cite{key400} \\
FGFR2 & amplification & \cite{key400} \\
\hline 
\end{tabular}
\end{table}



\subsection{Molecular Interaction}

\begin{enumerate}
	\item Level 1: DNA-DNA interaction 
	\item Level 2: DNA-protein interaction
	\item Level 3: DNA-Compound interaction
	\item Level 4: Protein-Compound interaction 
	\item Level 5: Protein-Protein interaction 
	\item Level 6: Compound-Compound interaction 
\end{enumerate}

\section{Mucin Structure}

\begin{table}[H]
	\caption{The Structure of Mucin in Gastric Cancer \cite{key400}}	
	\begin{tabular}{p{3cm}p{5cm}}
		\hline
		Protein & Observation & Reference \\
		\hline 
		MUC1, MUC4 & ERbB Interaction Mediates HER2, MAPK, PI3K and C-SRC & \cite{key400} \\
		MUC1 & Overexpression &  \cite{key400}  \\
		MUC1 & MUC4 silencing to Trastuzumab &  \cite{key400} \\
		MUC1  & Overexpression with several epithelial and transcriptional regulation with GC rich with Sp1, AP1, AP2 AP3, NF-1, ER and STAT transcription factors. &  \cite{key400} \\
		MUC1  & promoter contains 4 STAT binding and MUC1 promoter UP Regulation with INF-gamma and STAT-1,STAT-3 activation by IL-6. &  \cite{key400} \\
		MUC1  & Expression HIF-1 UP Regulation &  \cite{key400} \\
		MUC2,MUC5AC & Deregulated, Overexpression, Decreased ERK signaling &  \cite{key400} \\
		MUC2 &  5-FU (fluorouracil) Overexpression and methotrexate (chemo postive relationship with MUC2) &  \cite{key400} \\
		MUC4 &  Chemo Compound Resistence and Chemosensitization DOWN Regulation &  \cite{key400} \\
		MUC4 & PI3K/AKT, ERK1/2 and SRC/FAK &  \cite{key400} \\
		MUC5A &C Overexpression with airway inflammation &  \cite{key400} \\
		CA125/MUC16 &  EGFR Modulation &  \cite{key400} \\
		\hline 
	\end{tabular}
\end{table}

\section{Phytochemicals}

\begin{table}[H]
\caption{Category and Description of Phytochemicals}	
\begin{tabular}{p{3cm}p{3cm}}
\hline
Category & Description \\
\hline 
vitamins (carotenoids) & \\
polyphenols  & \\
flavonoids & \\
phytoalexins & \\
phenolic acids & \\
indoles  & \\
sulfur rich & \\  
\hline 
\end{tabular}
\end{table}

\section{Plant Components}

\begin{enumerate}
\item curcumin
\item Apigenin
\item Thymoquinone (TQ)
\item Silibinin
\item Scutellarin 
\item Quercetin
\item Resveratrol (3, 5, 4′-trihydroxy-trans-stilbene)
\item Ursolic acid, a pentacyclic triterpenoid
\end{enumerate}

\subsection{Plant: Roots}

\begin{enumerate}
\item Glycyrrhiza glabra
\end{enumerate}


\subsection{Plant: Leaves}

\begin{enumerate}
\item Epigallocatechin-3-gallate (EGCG)
\end{enumerate}


\subsection{Plant: Bark}

\begin{enumerate}
\item Gum Guggul
\end{enumerate}


\subsection{Plan of the Article}

\begin{enumerate}
\end{enumerate}



\section{Mathematical Background}

\subsection{Equation System}

\subsubsection{Equation System A}

\begin{align*} 
\tiny
\frac{d^{\alpha_1}X_1(t)}{dt^{\alpha_1}} = a_{11} *X_1(t - \tau_1) + \\
a_{12} *\frac{X_1(t)^{\beta_1}}{(1-X_1(t - \tau_1))^{\beta_1}} - a_{15}X_1(t) + \\
\epsilon_1(t) \\
\frac{d^{\alpha_1}X_2(t)}{dt^{\alpha_1}} = a_{21} *X_2(t - \tau_2) + \\
a_{22} *\frac{X_2(t)^{\beta_1}}{(1-X_2(t - \tau_2))^{\beta_1}} - a_{25}X_2(t) + \\
\epsilon_2(t) \\ \\
\frac{d^{\alpha_1}X_3(t)}{dt^{\alpha_1}} = a_{31} *X_3(t - \tau_3) + \\
a_{32} *\frac{X_3(t)^{\beta_1}}{(1-X_3(t - \tau_3))^{\beta_1}} - a_{35}X_3(t) + \\
\epsilon_3(t) \\ \\
\frac{d^{\alpha_1}X_4(t)}{dt^{\alpha_1}} = a_{41} *X_4(t - \tau_4) + \\
a_{42} *\frac{X_4(t)^{\beta_1}}{(1-X_4(t - \tau_4))^{\beta_1}} - a_{45}X_4(t) + \\
\epsilon_4(t) \\ \\
\frac{d^{\alpha_1}X_5(t)}{dt^{\alpha_1}} = a_{51} *X_5(t - \tau_5) + \\
a_{52} *\frac{X_5(t)^{\beta_1}}{(1-X_5(t - \tau_5))^{\beta_1}} - a_{55}X_5(t) + \\
\epsilon_5(t) \\
\end{align*}

\subsubsection{Parameter Table}

\begin{table}[h]\footnotesize
	\caption{Parameter Description and Value}
	\begin{tabular}{rllp{2cm}l}
		\hline	
		Parameter & Value & Interval & Description & Reference \\
		\hline 
		a11 & 0 & [0,1] & Equation 1 & \cite{key1}
		a12 & 0 & [0,1] & Equation 1 & \cite{key1}
		a13 & 0 & [0,1] & Equation 1 & \cite{key1}
		a14 & 0 & [0,1] & Equation 1 & \cite{key1}
		a15 & 0 & [0,1] & Equation 1 & \cite{key1}
		\hline
		a21 & 0 & [0,1] & Equation 2 & \cite{key1}
		a22 & 0 & [0,1] & Equation 2 & \cite{key1}
		a23 & 0 & [0,1] & Equation 2 & \cite{key1}
		a24 & 0 & [0,1] & Equation 2 & \cite{key1}
		a25 & 0 & [0,1] & Equation 2 & \cite{key1}
		\hline
		a31 & 0 & [0,1] & Equation 3 & \cite{key1}
		a32 & 0 & [0,1] & Equation 3 & \cite{key1}
		a33 & 0 & [0,1] & Equation 3 & \cite{key1}
		a34 & 0 & [0,1] & Equation 3 & \cite{key1}
		a35 & 0 & [0,1] & Equation 3 & \cite{key1}
		\hline
		a41 & 0 & [0,1] & Equation 4 & \cite{key1}
		a42 & 0 & [0,1] & Equation 4 & \cite{key1}
		a43 & 0 & [0,1] & Equation 4 & \cite{key1}
		a44 & 0 & [0,1] & Equation 4 & \cite{key1}
		a45 & 0 & [0,1] & Equation 4 & \cite{key1}
		\hline
		a51 & 0 & [0,1] & Equation 5 & \cite{key1}
		a52 & 0 & [0,1] & Equation 5 & \cite{key1}
		a53 & 0 & [0,1] & Equation 5 & \cite{key1}
		a54 & 0 & [0,1] & Equation 5 & \cite{key1}
		a55 & 0 & [0,1] & Equation 5 & \cite{key1}
		\hline
		$\tau_1$ & 1 & [1,2] & Equation 1 & \cite{key1}
		$\tau_2$ & 1 & [1,2] & Equation 2 & \cite{key1}
		$\tau_3$ & 1 & [1,2] & Equation 3 & \cite{key1}
		$\tau_4$ & 1 & [1,2] & Equation 4 & \cite{key1}
		$\tau_5$ & 1 & [1,2] & Equation 5 & \cite{key1}
		\hline
		$\alpha_1$ & 1 & (0,2] & Equation 1 & \cite{key1}
		$\alpha_2$ & 1 & (0,2] & Equation 2 & \cite{key1}
		$\alpha_3$ & 1 & (0,2] & Equation 3 & \cite{key1}
		$\alpha_4$ & 1 & (0,2] & Equation 4 & \cite{key1}
		$\alpha_5$ & 1 & (0,2] & Equation 5 & \cite{key1}
		\hline
		$\beta_1$ & 1 & (0,2] & Equation 1 & \cite{key1}
		$\beta_2$ & 1 & (0,2] & Equation 2 & \cite{key1}
		$\beta_3$ & 1 & (0,2] & Equation 3 & \cite{key1}
		$\beta_4$ & 1 & (0,2] & Equation 4 & \cite{key1}
		$\beta_5$ & 1 & (0,2] & Equation 5 & \cite{key1}
	\end{tabular}	
\end{table}



\subsection{Toplogical Dynamics}

\begin{table}[H]\centering
	\begin{tabular}{p{1cm}p{4cm}p{3cm}}
		Article ID & Summary & Comments\\
		\hline
		\hline
	\end{tabular}
\end{table}

\subsection{Phase Space Analysis}

\subsection{Stability}
\centering
\begin{table}[H]\footnotesize
	\caption{}
	\begin{tabular}{rp{1cm}p{2cm}p{3cm}p{1cm}}
		\hline
		ID & A & B & C & Reference \\
		\hline
		\hline
	\end{tabular}
\end{table}
\raggedright


\section{Results}


\subsection{Tables}

\centering
\begin{table}[H]\footnotesize
	\caption{}
	\begin{tabular}{rp{1cm}p{2cm}p{3cm}p{1cm}}
		\hline
		ID & A & B & C & Reference \\
		\hline
		\hline
	\end{tabular}
\end{table}
\raggedright

\centering
\begin{table}[H]\footnotesize
	\caption{}
	\begin{tabular}{rp{1cm}p{2cm}p{3cm}p{1cm}}
		\hline
		ID & A & B & C & Reference \\
		\hline
		\hline
	\end{tabular}
\end{table}
\raggedright

\centering
\begin{table}[H]\footnotesize
	\caption{}
	\begin{tabular}{rp{1cm}p{2cm}p{3cm}p{1cm}}
		\hline
		ID & A & B & C & Reference \\
		\hline
		\hline
	\end{tabular}
\end{table}
\raggedright


\subsection{Figures}

\begin{figure}[H]
	\centering
	\begin{minipage}[b]{0.5\linewidth}
	%\includegraphics[scale=0.25]{Example_1_Figure_1.png}
	\end{minipage}\hfill
	\begin{minipage}[b]{0.5\linewidth}
	%\includegraphics[scale=0.25]{Example_1_Figure_2.png}
	\end{minipage}\hfill	
	\begin{minipage}[b]{0.5\linewidth}
	%\includegraphics[scale=0.25]{Example_1_Figure_3.png}
	\end{minipage}\hfill
	\begin{minipage}[b]{0.5\linewidth}
	%\includegraphics[scale=0.25]{Example_1_Figure_4.png}
	\end{minipage}\hfill
	\caption{1, 2, 3 and 4}
	\label{fig:Figure1}
\end{figure} 



\section{Conclusions and Discussion Topics}

\begin{table}[H]\centering
	\begin{tabular}{p{1cm}p{4cm}p{3cm}}
		Topic ID & Summary & Comments\\
		\hline
		\hline
	\end{tabular}
\end{table}


\section{R Application Programming Interfaces (APIs)}



\bibliographystyle{plain}
\begin{thebibliography}{00}

\subsection{Main Classroom Article}

\bibitem[1]{key1}Emerging Potential of Natural Products for Targeting Mucins for Therapy Against Inflammation and Cancer

\bibitem[100]{key100}Cali, Yongli, Yun Kang, Weiming Wang (2017)
\newblock \textbf{A Stochastic SIRS epidemic model with nonlinear incidence rate}
\newblock Applied Mathematics and Computation 305: 221-240.

\bibitem[400]{key400} Kanehisa, Furumichi, M., Tanabe, M., Sato, Y., and Morishima, K.; 
\newblock KEGG: new perspectives on genomes, pathways, diseases and drugs. 
\newblock Nucleic Acids Res. 45, D353-D361 (2017).

\bibitem[401]{key401} Kanehisa, M., Sato, Y., Kawashima, M., Furumichi, M., and Tanabe, M.; 
\newblock KEGG as a reference resource for gene and protein annotation. 
\newblock Nucleic Acids Res. 44, D457-D462 (2016).

\bibitem[402]{key402} Kanehisa, M. and Goto, S.; 
\newblock KEGG: Kyoto Encyclopedia of Genes and Genomes. 
\newblock Nucleic Acids Res. 28, 27-30 (2000). 

\subsection{Stability and Convergence Analysis}

\bibitem[200]{key200}Bahar, Arifah and Mao, Xuerong (2004) 
\newblock Stochastic delay Lotka-Volterra model. 
\newblock Journal of Mathematical Analysis and Applications, 292 (2). pp. 364-380. ISSN 0022-247X , http://dx.doi.org/10.1016
/j.jmaa.2003.12.004

\bibitem[201]{key201} X.Li,X.Mao
\newblock Population dynamical behavior of non-autonomous Lotka–Volterra competitive system with random perturbation, 
\newblock Discret. Contin. Dyn. Syst. 24 (2) (2009) 523–593 .

\bibitem[202]{key202} M. Liu, M.Fan, 
\newblock Permanence of stochastic Lotka–Volterra systems,
\newblock J. Nonlinear Sci. (2016), doi:10.1007/s00332-016-9337-2.

\bibitem[203]{key203} A. Lahrouz, L. Omari,
\newblock Extinction and stationary distribution of a stochastic SIRS epidemic model with non-linear incidence,
\newblock Stat. Probab. Lett. 83 (2013) 960–968.

\bibitem[204]{key204} Q.Yang, X. Mao,
\newblock Stochastic dynamics of SIRS epidemic models with random perturbation,
\newblock Math. Biosci. Eng. 1

\bibitem[205]{key205} C. Xu
\newblock Global threshold dynamics of a stochastic differential equation SIS model
\newblock J. Math. Anal. Appl. 447 (2017) 736–757.

\bibitem[206]{key206}Arifah, B. and X. Mao 
\newblock Stochastic Delay Lotka-Volterra Model 
\newblock Journal of Math Anal App. 292 (2) (2004) 364-380.

\bibitem[207]{key207} D.Jiang,N.Shi,X.Li,
\newblock Global stability and stochastic permanence of a non-autonomous logistic equation with random perturbation
\newblock J.Math. Anal. Appl. 340 (1) (2008) 588–597

\bibitem[208]{key208}L.R. Bellet,
\newblock Ergodic properties of Markov processes, in: Open Quantum Systems II,
\newblock Springer, Berlin Heidelberg,2006, pp.1–39.

\bibitem[209]{key209} J. Cushing and J. Hudson, 2012. 
\newblock Evolutionary dynamics and strong Allee effects,
\newblock Journal  of  Biological Dynamics, 6 (2), 941-958.

\bibitem[210]{key210}H.W.Hethcote,
\newblock The mathematics of infectious diseases,
\newblock SIAM Rev. 42(4)(2000) 599–653.

\bibitem[211]{key211}B.Fred,C.-C.Carlos
\newblock Mathematical Models in Population Biology and Epidemiology,
\newblock second ed., Springer,2012.

\bibitem[1000]{key1000}R Core Team (2015). 
\newblock R: A language and environment for statistical computing. R Foundation for Statistical Computing, Vienna, Austria.
\newblock URL https://www.R-project.org/.

\end{thebibliography}
\end{document}
