%-------------------------------------------------------------------------%
%--------------------Classroom Lecture Model Series-----------------------%
%-------------------------------------------------------------------------%

\begin{document}
\twocolumn
\scriptsize
\begin{frontmatter}
		\title{DRAFT Sample: A Mathematical Model of DNA Repair Genes/Proteins Based on Molecular Function
and Signaling Pathway}
		\author{\corref{cor1}\fnref{fn1}}
		\cortext[cor1]{Corresponding author}
		\address{The Mathematical Learning Space}
		\ead{http://mathlearningspace.weebly.com}	
\end{frontmatter}	

Introduction:
\begin{enumerate}
\item Objective 1:
\item Objective 2:
\item Objective 3:
\end{enumerate}
Conclusion:

Keywords: 
Vocabulary Words:

\section{Introduction}

\centering
\begin{table}[H]\footnotesize
	\caption{}
	\begin{tabular}{rp{1cm}p{2cm}p{3cm}p{1cm}}
		\hline
		ID & A & B & C & Reference \\
		\hline
		\hline
	\end{tabular}
\end{table}
\raggedright


\subsection{Plan of the Article}

\begin{enumerate}
\end{enumerate}

\section{Topic Review Signal Network:}

\centering	
\begin{table}[H]\tiny
	\caption{}	
	\begin{tabular}{rp{1cm}|p{4cm}|l}
		\hline	
		PubmedID & Topic & Description & Citation \\
		\hline 
		\hline 
	\end{tabular}
\end{table}

\begin{table}[H]
\tiny
\begin{tabular}{p{1cm}p{1cm}p{3cm}p{1cm}p{1cm}} 
EdgeID & Gene 1 & Gene 2 & Biological Function & Comment \\
\hline
\hline
\end{tabular}
\caption{Signal Network}
\label{tab:Table2}
\end{table}

\begin{center}
	\begin{tikzpicture}
	[->,>=stealth',shorten >=2pt,node distance=3cm,
	thick,main node/.style={circle,draw,scale=0.25,transform canvas={scale=0.75},font=\sffamily\Small\bfseries},
	blacknode/.style={shape=circle, draw=black, line width=2},
	bluenode/.style={shape=circle, draw=blue, line width=2},
	greennode/.style={shape=circle, draw=green, line width=2},
	rednode/.style={shape=circle, draw=red, line width=2}
	]
%-------Legend ------------------------------------------------------------	
	\matrix [draw,below left] at (current bounding box.south) {
		\node [state,label=right:State] {}; A. \\
		\node [shapeSquare,label=right:Square- B.] {}; \\
		\node [shapeEllipse,label=right:Ellipse-C.] {}; \\
		\node [shapeTriangle,label=right:Triangle-D.] {}; \\
		\node [shapeHexagon,label=right:Hexagon-E.] {}; \\
	};
	\end{tikzpicture}
\end{center}

\section{Methods and Materials}

\subsection{Equation Systems}

Let X(t) be a vector of molecular entities such as genes/proteins/enzymes $X(t) = [x_{1}(t), x_{2}(t),...,x_{k}(t)]$ over the time period t=0,...,T where $t_{0}$ is the initial time period with $X(t_{0})$ for each molecular entity.  The fractional derivative $\frac{d^{\alpha} X(t)}{dt^{\alpha}}$ is the $\alpha$ derivative of X(t) such that $0 \le \alpha \le 2$ and $\tau$ delay parameter $X(t-\tau)$ in time represented by

\begin{equation}
\begin{cases}
\frac{d^{\alpha} X(t)}{dt^{\alpha}}=0 
\end{cases}
\end{equation}

For each $x_{i}(t)$ from 1 to K the following equations are specified.

\begin{equation}
\begin{cases}
\frac{d^{\alpha_{1}} x_{i}(t)}{dt^{\alpha_{1}}}=0 
\end{cases}
\end{equation}

\section{MAPK}

\subsection{Equation System A}

\begin{align*} 
\tiny
\frac{d^{\alpha_1}X_1(t)}{dt^{\alpha_1}} = a_{11} *X_1(t - \tau_1) + \\
a_{12} *\frac{X_1(t)^{\beta_1}}{(1-X_1(t - \tau_1))^{\beta_1}} - a_{15}X_1(t) + \\
\epsilon_1(t) \\
\frac{d^{\alpha_1}X_2(t)}{dt^{\alpha_1}} = a_{21} *X_2(t - \tau_2) + \\
a_{22} *\frac{X_2(t)^{\beta_1}}{(1-X_2(t - \tau_2))^{\beta_1}} - a_{25}X_2(t) + \\
\epsilon_2(t) \\ \\
\frac{d^{\alpha_1}X_3(t)}{dt^{\alpha_1}} = a_{31} *X_3(t - \tau_3) + \\
a_{32} *\frac{X_3(t)^{\beta_1}}{(1-X_3(t - \tau_3))^{\beta_1}} - a_{35}X_3(t) + \\
\epsilon_3(t) \\ \\
\frac{d^{\alpha_1}X_4(t)}{dt^{\alpha_1}} = a_{41} *X_4(t - \tau_4) + \\
a_{42} *\frac{X_4(t)^{\beta_1}}{(1-X_4(t - \tau_4))^{\beta_1}} - a_{45}X_4(t) + \\
\epsilon_4(t) \\ \\
\frac{d^{\alpha_1}X_5(t)}{dt^{\alpha_1}} = a_{51} *X_5(t - \tau_5) + \\
a_{52} *\frac{X_5(t)^{\beta_1}}{(1-X_5(t - \tau_5))^{\beta_1}} - a_{55}X_5(t) + \\
\epsilon_5(t) \\
\end{align*}

\subsubsection{Parameter Table}

\begin{table}[h]\footnotesize
	\caption{Parameter Description and Value}
	\begin{tabular}{rllp{2cm}l}
		\hline	
		Parameter & Value & Interval & Description & Reference \\
		\hline 
		a11 & 0 & [0,1] & Equation 1 & \cite{key1}
		a12 & 0 & [0,1] & Equation 1 & \cite{key1}
		a13 & 0 & [0,1] & Equation 1 & \cite{key1}
		a14 & 0 & [0,1] & Equation 1 & \cite{key1}
		a15 & 0 & [0,1] & Equation 1 & \cite{key1}
		\hline
		a21 & 0 & [0,1] & Equation 2 & \cite{key1}
		a22 & 0 & [0,1] & Equation 2 & \cite{key1}
		a23 & 0 & [0,1] & Equation 2 & \cite{key1}
		a24 & 0 & [0,1] & Equation 2 & \cite{key1}
		a25 & 0 & [0,1] & Equation 2 & \cite{key1}
		\hline
		a31 & 0 & [0,1] & Equation 3 & \cite{key1}
		a32 & 0 & [0,1] & Equation 3 & \cite{key1}
		a33 & 0 & [0,1] & Equation 3 & \cite{key1}
		a34 & 0 & [0,1] & Equation 3 & \cite{key1}
		a35 & 0 & [0,1] & Equation 3 & \cite{key1}
		\hline
		a41 & 0 & [0,1] & Equation 4 & \cite{key1}
		a42 & 0 & [0,1] & Equation 4 & \cite{key1}
		a43 & 0 & [0,1] & Equation 4 & \cite{key1}
		a44 & 0 & [0,1] & Equation 4 & \cite{key1}
		a45 & 0 & [0,1] & Equation 4 & \cite{key1}
		\hline
		a51 & 0 & [0,1] & Equation 5 & \cite{key1}
		a52 & 0 & [0,1] & Equation 5 & \cite{key1}
		a53 & 0 & [0,1] & Equation 5 & \cite{key1}
		a54 & 0 & [0,1] & Equation 5 & \cite{key1}
		a55 & 0 & [0,1] & Equation 5 & \cite{key1}
		\hline
		$\tau_1$ & 1 & [1,2] & Equation 1 & \cite{key1}
		$\tau_2$ & 1 & [1,2] & Equation 2 & \cite{key1}
		$\tau_3$ & 1 & [1,2] & Equation 3 & \cite{key1}
		$\tau_4$ & 1 & [1,2] & Equation 4 & \cite{key1}
		$\tau_5$ & 1 & [1,2] & Equation 5 & \cite{key1}
		\hline
		$\alpha_1$ & 1 & (0,2] & Equation 1 & \cite{key1}
		$\alpha_2$ & 1 & (0,2] & Equation 2 & \cite{key1}
		$\alpha_3$ & 1 & (0,2] & Equation 3 & \cite{key1}
		$\alpha_4$ & 1 & (0,2] & Equation 4 & \cite{key1}
		$\alpha_5$ & 1 & (0,2] & Equation 5 & \cite{key1}
		\hline
		$\beta_1$ & 1 & (0,2] & Equation 1 & \cite{key1}
		$\beta_2$ & 1 & (0,2] & Equation 2 & \cite{key1}
		$\beta_3$ & 1 & (0,2] & Equation 3 & \cite{key1}
		$\beta_4$ & 1 & (0,2] & Equation 4 & \cite{key1}
		$\beta_5$ & 1 & (0,2] & Equation 5 & \cite{key1}
	\end{tabular}	
\end{table}




\section{Calcium} 

\subsection{Equation System A}

\begin{align*} 
\tiny
\frac{d^{\alpha_1}X_1(t)}{dt^{\alpha_1}} = a_{11} *X_1(t - \tau_1) + \\
a_{12} *\frac{X_1(t)^{\beta_1}}{(1-X_1(t - \tau_1))^{\beta_1}} - a_{15}X_1(t) + \\
\epsilon_1(t) \\
\frac{d^{\alpha_1}X_2(t)}{dt^{\alpha_1}} = a_{21} *X_2(t - \tau_2) + \\
a_{22} *\frac{X_2(t)^{\beta_1}}{(1-X_2(t - \tau_2))^{\beta_1}} - a_{25}X_2(t) + \\
\epsilon_2(t) \\ \\
\frac{d^{\alpha_1}X_3(t)}{dt^{\alpha_1}} = a_{31} *X_3(t - \tau_3) + \\
a_{32} *\frac{X_3(t)^{\beta_1}}{(1-X_3(t - \tau_3))^{\beta_1}} - a_{35}X_3(t) + \\
\epsilon_3(t) \\ \\
\frac{d^{\alpha_1}X_4(t)}{dt^{\alpha_1}} = a_{41} *X_4(t - \tau_4) + \\
a_{42} *\frac{X_4(t)^{\beta_1}}{(1-X_4(t - \tau_4))^{\beta_1}} - a_{45}X_4(t) + \\
\epsilon_4(t) \\ \\
\frac{d^{\alpha_1}X_5(t)}{dt^{\alpha_1}} = a_{51} *X_5(t - \tau_5) + \\
a_{52} *\frac{X_5(t)^{\beta_1}}{(1-X_5(t - \tau_5))^{\beta_1}} - a_{55}X_5(t) + \\
\epsilon_5(t) \\
\end{align*}

\subsubsection{Parameter Table}

\begin{table}[h]\footnotesize
	\caption{Parameter Description and Value}
	\begin{tabular}{rllp{2cm}l}
		\hline	
		Parameter & Value & Interval & Description & Reference \\
		\hline 
		a11 & 0 & [0,1] & Equation 1 & \cite{key1}
		a12 & 0 & [0,1] & Equation 1 & \cite{key1}
		a13 & 0 & [0,1] & Equation 1 & \cite{key1}
		a14 & 0 & [0,1] & Equation 1 & \cite{key1}
		a15 & 0 & [0,1] & Equation 1 & \cite{key1}
		\hline
		a21 & 0 & [0,1] & Equation 2 & \cite{key1}
		a22 & 0 & [0,1] & Equation 2 & \cite{key1}
		a23 & 0 & [0,1] & Equation 2 & \cite{key1}
		a24 & 0 & [0,1] & Equation 2 & \cite{key1}
		a25 & 0 & [0,1] & Equation 2 & \cite{key1}
		\hline
		a31 & 0 & [0,1] & Equation 3 & \cite{key1}
		a32 & 0 & [0,1] & Equation 3 & \cite{key1}
		a33 & 0 & [0,1] & Equation 3 & \cite{key1}
		a34 & 0 & [0,1] & Equation 3 & \cite{key1}
		a35 & 0 & [0,1] & Equation 3 & \cite{key1}
		\hline
		a41 & 0 & [0,1] & Equation 4 & \cite{key1}
		a42 & 0 & [0,1] & Equation 4 & \cite{key1}
		a43 & 0 & [0,1] & Equation 4 & \cite{key1}
		a44 & 0 & [0,1] & Equation 4 & \cite{key1}
		a45 & 0 & [0,1] & Equation 4 & \cite{key1}
		\hline
		a51 & 0 & [0,1] & Equation 5 & \cite{key1}
		a52 & 0 & [0,1] & Equation 5 & \cite{key1}
		a53 & 0 & [0,1] & Equation 5 & \cite{key1}
		a54 & 0 & [0,1] & Equation 5 & \cite{key1}
		a55 & 0 & [0,1] & Equation 5 & \cite{key1}
		\hline
		$\tau_1$ & 1 & [1,2] & Equation 1 & \cite{key1}
		$\tau_2$ & 1 & [1,2] & Equation 2 & \cite{key1}
		$\tau_3$ & 1 & [1,2] & Equation 3 & \cite{key1}
		$\tau_4$ & 1 & [1,2] & Equation 4 & \cite{key1}
		$\tau_5$ & 1 & [1,2] & Equation 5 & \cite{key1}
		\hline
		$\alpha_1$ & 1 & (0,2] & Equation 1 & \cite{key1}
		$\alpha_2$ & 1 & (0,2] & Equation 2 & \cite{key1}
		$\alpha_3$ & 1 & (0,2] & Equation 3 & \cite{key1}
		$\alpha_4$ & 1 & (0,2] & Equation 4 & \cite{key1}
		$\alpha_5$ & 1 & (0,2] & Equation 5 & \cite{key1}
		\hline
		$\beta_1$ & 1 & (0,2] & Equation 1 & \cite{key1}
		$\beta_2$ & 1 & (0,2] & Equation 2 & \cite{key1}
		$\beta_3$ & 1 & (0,2] & Equation 3 & \cite{key1}
		$\beta_4$ & 1 & (0,2] & Equation 4 & \cite{key1}
		$\beta_5$ & 1 & (0,2] & Equation 5 & \cite{key1}
	\end{tabular}	
\end{table}



\section{cAMP}


\subsection{Equation System A}

\begin{align*} 
\tiny
\frac{d^{\alpha_1}X_1(t)}{dt^{\alpha_1}} = a_{11} *X_1(t - \tau_1) + \\
a_{12} *\frac{X_1(t)^{\beta_1}}{(1-X_1(t - \tau_1))^{\beta_1}} - a_{15}X_1(t) + \\
\epsilon_1(t) \\
\frac{d^{\alpha_1}X_2(t)}{dt^{\alpha_1}} = a_{21} *X_2(t - \tau_2) + \\
a_{22} *\frac{X_2(t)^{\beta_1}}{(1-X_2(t - \tau_2))^{\beta_1}} - a_{25}X_2(t) + \\
\epsilon_2(t) \\ \\
\frac{d^{\alpha_1}X_3(t)}{dt^{\alpha_1}} = a_{31} *X_3(t - \tau_3) + \\
a_{32} *\frac{X_3(t)^{\beta_1}}{(1-X_3(t - \tau_3))^{\beta_1}} - a_{35}X_3(t) + \\
\epsilon_3(t) \\ \\
\frac{d^{\alpha_1}X_4(t)}{dt^{\alpha_1}} = a_{41} *X_4(t - \tau_4) + \\
a_{42} *\frac{X_4(t)^{\beta_1}}{(1-X_4(t - \tau_4))^{\beta_1}} - a_{45}X_4(t) + \\
\epsilon_4(t) \\ \\
\frac{d^{\alpha_1}X_5(t)}{dt^{\alpha_1}} = a_{51} *X_5(t - \tau_5) + \\
a_{52} *\frac{X_5(t)^{\beta_1}}{(1-X_5(t - \tau_5))^{\beta_1}} - a_{55}X_5(t) + \\
\epsilon_5(t) \\
\end{align*}


\subsubsection{Parameter Table}

\begin{table}[h]\footnotesize
	\caption{Parameter Description and Value}
	\begin{tabular}{rllp{2cm}l}
		\hline	
		Parameter & Value & Interval & Description & Reference \\
		\hline 
		a11 & 0 & [0,1] & Equation 1 & \cite{key1}
		a12 & 0 & [0,1] & Equation 1 & \cite{key1}
		a13 & 0 & [0,1] & Equation 1 & \cite{key1}
		a14 & 0 & [0,1] & Equation 1 & \cite{key1}
		a15 & 0 & [0,1] & Equation 1 & \cite{key1}
		\hline
		a21 & 0 & [0,1] & Equation 2 & \cite{key1}
		a22 & 0 & [0,1] & Equation 2 & \cite{key1}
		a23 & 0 & [0,1] & Equation 2 & \cite{key1}
		a24 & 0 & [0,1] & Equation 2 & \cite{key1}
		a25 & 0 & [0,1] & Equation 2 & \cite{key1}
		\hline
		a31 & 0 & [0,1] & Equation 3 & \cite{key1}
		a32 & 0 & [0,1] & Equation 3 & \cite{key1}
		a33 & 0 & [0,1] & Equation 3 & \cite{key1}
		a34 & 0 & [0,1] & Equation 3 & \cite{key1}
		a35 & 0 & [0,1] & Equation 3 & \cite{key1}
		\hline
		a41 & 0 & [0,1] & Equation 4 & \cite{key1}
		a42 & 0 & [0,1] & Equation 4 & \cite{key1}
		a43 & 0 & [0,1] & Equation 4 & \cite{key1}
		a44 & 0 & [0,1] & Equation 4 & \cite{key1}
		a45 & 0 & [0,1] & Equation 4 & \cite{key1}
		\hline
		a51 & 0 & [0,1] & Equation 5 & \cite{key1}
		a52 & 0 & [0,1] & Equation 5 & \cite{key1}
		a53 & 0 & [0,1] & Equation 5 & \cite{key1}
		a54 & 0 & [0,1] & Equation 5 & \cite{key1}
		a55 & 0 & [0,1] & Equation 5 & \cite{key1}
		\hline
		$\tau_1$ & 1 & [1,2] & Equation 1 & \cite{key1}
		$\tau_2$ & 1 & [1,2] & Equation 2 & \cite{key1}
		$\tau_3$ & 1 & [1,2] & Equation 3 & \cite{key1}
		$\tau_4$ & 1 & [1,2] & Equation 4 & \cite{key1}
		$\tau_5$ & 1 & [1,2] & Equation 5 & \cite{key1}
		\hline
		$\alpha_1$ & 1 & (0,2] & Equation 1 & \cite{key1}
		$\alpha_2$ & 1 & (0,2] & Equation 2 & \cite{key1}
		$\alpha_3$ & 1 & (0,2] & Equation 3 & \cite{key1}
		$\alpha_4$ & 1 & (0,2] & Equation 4 & \cite{key1}
		$\alpha_5$ & 1 & (0,2] & Equation 5 & \cite{key1}
		\hline
		$\beta_1$ & 1 & (0,2] & Equation 1 & \cite{key1}
		$\beta_2$ & 1 & (0,2] & Equation 2 & \cite{key1}
		$\beta_3$ & 1 & (0,2] & Equation 3 & \cite{key1}
		$\beta_4$ & 1 & (0,2] & Equation 4 & \cite{key1}
		$\beta_5$ & 1 & (0,2] & Equation 5 & \cite{key1}
	\end{tabular}	
\end{table}



\subsection{Algorithms}

\begin{algorithm}[H]
	\footnotesize
	\begin{algorithmic}[1]
		\Return $Z$
	\end{algorithmic}
	\caption{}\label{Algorithm_1}
\end{algorithm}

\subsection{Tables}

\subsection{Initial Conditions, Parameter Matrices, and Noise Distributions}

\subsubsection{MAPK} 

\subsection{Parameter Matrix}
\vspace{4pt}
\centering
\begin{table}[H]\footnotesize
\caption{Parameter Description and Value}
\begin{tabular}{rllp{2cm}l}
	\hline	
	Parameter & Value & Interval & Description & Reference \\
	\hline 
\end{tabular}	
\end{table}

\subsubsection{Calcium} 

\subsection{Parameter Matrix}
\vspace{4pt}
\centering
\begin{table}[H]\footnotesize
\caption{Parameter Description and Value}
\begin{tabular}{rllp{2cm}l}
	\hline	
	Parameter & Value & Interval & Description & Reference \\
	\hline 
\end{tabular}	
\end{table}

\subsubsection{cAMP}

\subsection{Parameter Matrix}
\vspace{4pt}
\centering
\begin{table}[H]\footnotesize
\caption{Parameter Description and Value}
\begin{tabular}{rllp{2cm}l}
	\hline	
	Parameter & Value & Interval & Description & Reference \\
	\hline 
\end{tabular}	
\end{table}

\subsection{Figures}

\begin{figure}[H]
	\centering
	\begin{minipage}[b]{0.5\linewidth}
	%\includegraphics[scale=0.25]{Example_1_Figure_1.png}
	\end{minipage}\hfill
	\begin{minipage}[b]{0.5\linewidth}
	%\includegraphics[scale=0.25]{Example_1_Figure_2.png}
	\end{minipage}\hfill	
	\begin{minipage}[b]{0.5\linewidth}
	%\includegraphics[scale=0.25]{Example_1_Figure_3.png}
	\end{minipage}\hfill
	\begin{minipage}[b]{0.5\linewidth}
	%\includegraphics[scale=0.25]{Example_1_Figure_4.png}
	\end{minipage}\hfill
	\caption{1, 2, 3 and 4}
	\label{fig:Figure1}
\end{figure} 


\section{Conclusions}

\begin{enumerate}
\end{enumerate}


\section{Topics for Classroom Discussion}

\begin{table}[H]\centering
	\begin{tabular}{p{1cm}p{4cm}p{3cm}}
		Article ID & Summary & Comments\\
		\hline
		\hline
	\end{tabular}
\end{table}



\section{R Application Programming Interfaces (APIs)}




\bibliographystyle{plain}
\begin{thebibliography}{00}

\footnotesize
\bibitem[1]{key1}Li H1, Zhang XP, Liu F. (2013). 
\newblock Coordination between p21 and DDB2 in the cellular response to UV radiation 
\newblock PLoS One. https://www.ncbi.nlm.nih.gov/pubmed/24260342

\bibitem[2]{key20}Fitch ME, Nakajima S, Yasui A, Ford JM (2003) 
\newblock In vivo recruitment of XPC to UV-induced cyclobutane pyrimidine dimers by the DDB2 gene product. 
\newblock J Biol Chem 278: 46906–46910.

\bibitem[3]{key28} Liu S, Shiotani B, Lahiri M, Marechal A, Tse A, et al. (2011) 
\newblock ATR autophosphorylation as a molecular switch for checkpoint activation. 
\newblock Mol Cell 43: 192–202.

\bibitem[4]{key23} Stoyanova T, Roy N, Kopanja D, Bagchi S, Raychaudhuri P (2009) 
\newblock DDB2 decides cell fate following DNA damage. 
\newblock Proc Natl Acad Sci USA 106: 10690–10695.

\bibitem[5]{key11} Abbas T, Dutta A (2009) 
\newblock p21 in cancer: intricate networks and multiple activities. 
\newblock Nat Rev Cancer 9: 400–414.

\bibitem[6]{key34} Mayo LD, Donner DB (2002) 
\newblock The PTEN, Mdm2, p53 tumor suppressor-oncoprotein network. 
\newblock Trends Biochem Sci 27: 462–467.

\bibitem[7]{key39} 39. Song MS, Salmena L, Pandolfi PP (2012) 
\newblock The functions and regulation of the PTEN tumour suppressor. 
\newblock Nat Rev Mol Cell Biol 13: 283–296.

\bibitem[8]{key40} McCubrey JA, Steelman LS, Kempf CR, Chappell WH, Abrams SL, et al. (2011) 
\newblock Therapeutic resistance resulting from mutations in Raf/MEK/ERK and PI3K/PTEN/Akt/mTOR signaling pathways. 
\newblock J Cell Physiol 226: 2762–2781.

\bibitem[9]{key33}. Wu X, Bayle JH, Olson D, Levine AJ (1993) 
\newblock The p53-mdm-2 autoregulatory feedback loop. 
\newblock Genes Dev 7: 1126–1132.

\bibitem[10]{key35}. Stambolic V, MacPherson D, Sas D, Lin Y, Snow B, et al. (2001) 
\newblock Regulation of PTEN transcription by p53. 
\newblock Mol Cell 8: 317–325.

\bibitem[11]{key36}. Stambolic V, Suzuki A, de la Pompa JL, Brothers GM, Mirtsos C, et al. (1998)
\newblock Negative regulation of PKB/Akt-dependent cell survival by the tumor suppressor 
\newblock PTEN. Cell 95: 29–39.

\bibitem[12]{key52}Caldecott KW. 
\newblock Single-strand break repair and genetic disease. 
\newblock Nat Rev Genet. 2008; 9:619–631.[PubMed: 18626472]

\bibitem[13]{key53}Amé JC, Rolli V, Schreiber V, et al. 
\newblock PARP-2, A novel mammalian DNA damage-dependent poly(ADP-ribose) polymerase. 
\newblock J Biol Chem. 1999; 274:17860–17868. [PubMed: 10364231]

\bibitem[14]{key54}Ménissier de Murcia J, Ricoul M, Tartier L, et al. 
\newblock Functional interaction between PARP-1 and PARP-2 in chromosome stability and embryonic development in mouse. 
\newblock EMBOJ. 2003; 22:2255–2263.Tomkinson et al.

\bibitem[15]{key55} Lin, Y., Bai, L., Cupello, S., Hossain, M. A., Deem, B., McLeod, M., … Yan, S. (2018). 
\newblock APE2 promotes DNA damage response pathway from a single-strand break.
\newblock Nucleic Acids Research, 46(5), 2479–2494. http://doi.org/10.1093/nar/gky020

\bibitem[16]{key56} Falck, J., Mailand, N., Syljuasen, R. G., Bartek, J., Lukas, J. 
\newblock The ATM-Chk2-Cdc25A checkpoint pathway guards against radioresistant DNA synthesis. 
\newblock Nature 410: 842-847, 2001. 

\bibitem[17]{key57}Martin, G. A., Bollag, G., McCormick, F., Abo, A. 
\newblock A novel serine kinase activated by rac1/CDC42Hs-dependent autophosphorylation is related to PAK65 and STE20. 
\newblock EMBO J. 14: 1970-1978, 1995. Note: Erratum: EMBO J. 14: 4385 only, 1995. 

\bibitem[18]{key58}Jiang, X., Kim, H.-E., Shu, H., Zhao, Y., Zhang, H., Kofron, J., Donnelly, J., Burns, D., Ng, S., Rosenberg, S., Wang, X. 
\newblock Distinctive roles of PHAP proteins and prothymosin-alpha in a death regulatory pathway. 
\newblock Science 299: 223-226, 2003.

\bibitem[19]{key59}Hollander MC, Blumenthal GM, Dennis PA. 
\newblock PTEN loss in the continuum of common cancers, rare syndromes and mouse models.
\newblock Journal Nat Rev Cancer 11:289-301 (2011)

\bibitem[19]{key60}Li, D.-M., Sun, H. 
\newblock PTEN/MMAC1/TEP1 suppresses the tumorigenicity and induces G1 cell cycle arrest in human glioblastoma cells. 
\newblock Proc. Nat. Acad. Sci. 95: 15406-15411, 1998. 

\bibitem[20]{key61} KH, Sobol RW.
\newblock A unified view of base excision repair: lesion-dependent protein complexes regulated by post-translational modification.
\newblock DNA Repair (Amst) 6:695-711 (2007) DOI:10.1016/j.dnarep.2007.01.009 PMID:17337257

\bibitem[21]{key62} Rouillard AD, Gundersen GW, Fernandez NF, Wang Z, Monteiro CD, McDermott MG, Ma'ayan A. 
\newblock The harmonizome: a collection of processed datasets gathered to serve and mine knowledge about genes and proteins. 
\newblock Database (Oxford). 2016 Jul 3;2016. pii: baw100. 

\subsection{Differential Equations}

\bibitem[1]{key1}A New Approach and Solution Technique to Solve Time Fractional Nonlinear Reaction-Diffusion Equations
\bibitem[1]{key1}Stability Analysis of Fractional-Order Nonlinear Systems with Delay
\bibitem[1]{key1}Application of the Multistep Generalized Differential Transform Method to Solve a Time-Fractional Enzyme Kinetics
\bibitem[1]{key1}Wavelet Methods for Solving Fractional Order Differential Equations
\bibitem[1]{key1}Numerical Methods for Pricing American Options with Time-Fractional PDE Models
\bibitem[1]{key1}Application of Multistep Generalized Differential Transform Method for the Solutions of the Fractional-Order Chua System
\bibitem[1]{key1}Numerical Solution of Some Types of Fractional Optimal Control Problems
\bibitem[1]{key1}An Efficient Series Solution for Fractional Differential Equations
\bibitem[1]{key1}Approximate Analytical Solution for Nonlinear System of Fractional Differential Equations by BPs Operational Matrices
\bibitem[1]{key1}Numerical Solution for Complex Systems of Fractional Order
\bibitem[1]{key1}Stability Analysis of Fractional-Order Nonlinear Systems with Delay
\bibitem[1]{key1}Numerical Study for Time Delay Multistrain Tuberculosis Model of Fractional Order
\bibitem[1]{key1}A Numerical Method for Solving Fractional Differential Equations by Using Neural Network
\bibitem[1]{key1}Numerical Studies for Fractional-Order Logistic Differential Equation with Two Different Delays
\bibitem[1]{key1}Numerical Modeling of Fractional-Order Biological Systems
\bibitem[1]{key1}Numerical Solution of Some Types of Fractional Optimal Control Problems
\bibitem[1]{key1}A Numerical Method for Delayed Fractional-Order Differential Equations
\bibitem[1]{key1} New Insights into the Fractional Order Diffusion Equation Using Entropy and Kurtosis
\bibitem[1]{key1} DELAY DIFFERENTIAL EQUATIONS IN SINGLE SPECIES DYNAMICS
\bibitem[1]{key1} An Improved Artificial Bee Colony Algorithm Based on Elite Strategy and Dimension Learning
\bibitem[1]{key1}Operators of Fractional Calculus and Their Applications
\bibitem[1]{key1} Modelling Physiological and Pharmacological Control on Cell Proliferation to Optimise Cancer Treatments

\bibitem[1]{key1}Press, W. H., S. A. Teukolsky, W. T Vetterling, and B. P. Flannery (2007). 
\newblock Numerical Recipes: The Art of Numerical Computing. 
\newblock Third Edition, Cambridge University Press, New York.

\bibitem[400]{key400} Kanehisa, Furumichi, M., Tanabe, M., Sato, Y., and Morishima, K.; 
\newblock KEGG: new perspectives on genomes, pathways, diseases and drugs. 
\newblock Nucleic Acids Res. 45, D353-D361 (2017).

\bibitem[401]{key401} Kanehisa, M., Sato, Y., Kawashima, M., Furumichi, M., and Tanabe, M.; 
\newblock KEGG as a reference resource for gene and protein annotation. 
\newblock Nucleic Acids Res. 44, D457-D462 (2016).

\bibitem[402]{key402} Kanehisa, M. and Goto, S.; 
\newblock KEGG: Kyoto Encyclopedia of Genes and Genomes. 
\newblock Nucleic Acids Res. 28, 27-30 (2000). 

\bibitem[403]{key403} Rouillard AD, Gundersen GW, Fernandez NF, Wang Z, Monteiro CD, McDermott MG, Ma'ayan A. 
\newblock The harmonizome: a collection of processed datasets gathered to serve and mine knowledge about genes and proteins. 
\newblock Database (Oxford). 2016 Jul 3;2016. pii: baw100. 

\end{thebibliography}
\end{document}
