%-------------------------------------------------------------------------%
%--------------------Classroom Lecture Model Series-----------------------%
%-------------------------------------------------------------------------%

\begin{document}
\twocolumn
\scriptsize
\begin{frontmatter}
		\title{DRAFT Sample: A Mathematical model of Cell Cycle Process Regulation of Cell Cycle, Chromosome Segregation and G2/M Transition}
		\author{\corref{cor1}\fnref{fn1}}
		\cortext[cor1]{Corresponding author}
		\address{The Mathematical Learning Space}
		\ead{http://mathlearningspace.weebly.com}	
\end{frontmatter}	

Introduction:
\begin{enumerate}
\item Objective 1:
\item Objective 2:
\item Objective 3:
\end{enumerate}
Conclusion:

Keywords: KEGG Database, NCI-60 Cell lines, Gene Ontology, NCI Ontology
Vocabulary Words:

\section{Introduction}

\subsection{Plan of the Article}

\begin{enumerate}
\end{enumerate}

\section{Topic Review}

\subsection{Protein Interaction Diagram and Annotations}

\begin{tikzpicture}
	[->,>=stealth',shorten >=2pt,node distance=3cm,
	thick,main node/.style={circle,draw,scale=0.25,transform canvas={scale=0.75},font=\sffamily\Small\bfseries},
	blacknode/.style={shape=circle, draw=black, line width=2},
	bluenode/.style={shape=circle, draw=blue, line width=2},
	greennode/.style={shape=circle, draw=green, line width=2},
	rednode/.style={shape=circle, draw=red, line width=2}
	]
	%-------Legend ------------------------------------------------------------	
	\matrix [draw,below left] at (current bounding box.south) {
		\node [state,label=right:State] {}; Description. \\
		\node [shapeSquare,label=right:Square-.] {}; \\
		\node [shapeEllipse,label=right:Ellipse-.] {}; \\
		\node [shapeTriangle,label=right:Triangle-.] {}; \\
		\node [shapeHexagon,label=right:Hexagon-.] {}; \\
	};
\end{tikzpicture}

\centering	
\begin{table}[H]\tiny
\caption{Gene/Protein/Enzymes Description and References}	
\begin{tabular}{r|p{3cm}|l}
\hline	
Gene/Protein & Description & Reference \\
\hline 
\hline 
\end{tabular}
\end{table}

\subsection{Topic A: KEGG Database}

\centering	
\begin{table}[H]\tiny
	\caption{}	
	\begin{tabular}{p{1cm}p{1cm}|p{4cm}|l}
		\hline	
		PubMed ID & Topic & Description & Results \\
		\hline 
		\hline 
	\end{tabular}
\end{table}

\subsection{Topic B: NCI- 60 Cell Lines}

\subsection{Topic C: Gene Ontology}

\subsection{Topic D: NCI Ontology}

\section{Protein Interaction Diagram and Annotations}

\begin{tikzpicture}
	[->,>=stealth',shorten >=2pt,node distance=3cm,
	thick,main node/.style={circle,draw,scale=0.25,transform canvas={scale=0.75},font=\sffamily\Small\bfseries},
	blacknode/.style={shape=circle, draw=black, line width=2},
	bluenode/.style={shape=circle, draw=blue, line width=2},
	greennode/.style={shape=circle, draw=green, line width=2},
	rednode/.style={shape=circle, draw=red, line width=2}
	]
	%-------Legend ------------------------------------------------------------	
	\matrix [draw,below left] at (current bounding box.south) {
		\node [state,label=right:State] {}; Description. \\
		\node [shapeSquare,label=right:Square-.] {}; \\
		\node [shapeEllipse,label=right:Ellipse-.] {}; \\
		\node [shapeTriangle,label=right:Triangle-.] {}; \\
		\node [shapeHexagon,label=right:Hexagon-.] {}; \\
	};
\end{tikzpicture}

\centering	
\begin{table}[H]\tiny
\caption{Gene/Protein/Enzymes Description and References}	
\begin{tabular}{r|p{3cm}|l}
\hline	
Gene/Protein & Description & Reference \\
\hline 
\hline 
\end{tabular}
\end{table}

\section{Mathematical Model}

\subsection{Equations}

\subsection{Equation System A}

\begin{align*} 
\tiny
\frac{d^{\alpha_1}X_1(t)}{dt^{\alpha_1}} = a_{11} *X_1(t - \tau_1) + \\
a_{12} *\frac{X_1(t)^{\beta_1}}{(1-X_1(t - \tau_1))^{\beta_1}} - a_{15}X_1(t) + \\
\epsilon_1(t) \\
\frac{d^{\alpha_1}X_2(t)}{dt^{\alpha_1}} = a_{21} *X_2(t - \tau_2) + \\
a_{22} *\frac{X_2(t)^{\beta_1}}{(1-X_2(t - \tau_2))^{\beta_1}} - a_{25}X_2(t) + \\
\epsilon_2(t) \\ \\
\frac{d^{\alpha_1}X_3(t)}{dt^{\alpha_1}} = a_{31} *X_3(t - \tau_3) + \\
a_{32} *\frac{X_3(t)^{\beta_1}}{(1-X_3(t - \tau_3))^{\beta_1}} - a_{35}X_3(t) + \\
\epsilon_3(t) \\ \\
\frac{d^{\alpha_1}X_4(t)}{dt^{\alpha_1}} = a_{41} *X_4(t - \tau_4) + \\
a_{42} *\frac{X_4(t)^{\beta_1}}{(1-X_4(t - \tau_4))^{\beta_1}} - a_{45}X_4(t) + \\
\epsilon_4(t) \\ \\
\frac{d^{\alpha_1}X_5(t)}{dt^{\alpha_1}} = a_{51} *X_5(t - \tau_5) + \\
a_{52} *\frac{X_5(t)^{\beta_1}}{(1-X_5(t - \tau_5))^{\beta_1}} - a_{55}X_5(t) + \\
\epsilon_5(t) \\
\end{align*}

\subsubsection{Parameter Table}

\begin{table}[h]\footnotesize
	\caption{Parameter Description and Value}
	\begin{tabular}{rllp{2cm}l}
		\hline	
		Parameter & Value & Interval & Description & Reference \\
		\hline 
		a11 & 0 & [0,1] & Equation 1 & \cite{key1}
		a12 & 0 & [0,1] & Equation 1 & \cite{key1}
		a13 & 0 & [0,1] & Equation 1 & \cite{key1}
		a14 & 0 & [0,1] & Equation 1 & \cite{key1}
		a15 & 0 & [0,1] & Equation 1 & \cite{key1}
		\hline
		a21 & 0 & [0,1] & Equation 2 & \cite{key1}
		a22 & 0 & [0,1] & Equation 2 & \cite{key1}
		a23 & 0 & [0,1] & Equation 2 & \cite{key1}
		a24 & 0 & [0,1] & Equation 2 & \cite{key1}
		a25 & 0 & [0,1] & Equation 2 & \cite{key1}
		\hline
		a31 & 0 & [0,1] & Equation 3 & \cite{key1}
		a32 & 0 & [0,1] & Equation 3 & \cite{key1}
		a33 & 0 & [0,1] & Equation 3 & \cite{key1}
		a34 & 0 & [0,1] & Equation 3 & \cite{key1}
		a35 & 0 & [0,1] & Equation 3 & \cite{key1}
		\hline
		a41 & 0 & [0,1] & Equation 4 & \cite{key1}
		a42 & 0 & [0,1] & Equation 4 & \cite{key1}
		a43 & 0 & [0,1] & Equation 4 & \cite{key1}
		a44 & 0 & [0,1] & Equation 4 & \cite{key1}
		a45 & 0 & [0,1] & Equation 4 & \cite{key1}
		\hline
		a51 & 0 & [0,1] & Equation 5 & \cite{key1}
		a52 & 0 & [0,1] & Equation 5 & \cite{key1}
		a53 & 0 & [0,1] & Equation 5 & \cite{key1}
		a54 & 0 & [0,1] & Equation 5 & \cite{key1}
		a55 & 0 & [0,1] & Equation 5 & \cite{key1}
		\hline
		$\tau_1$ & 1 & [1,2] & Equation 1 & \cite{key1}
		$\tau_2$ & 1 & [1,2] & Equation 2 & \cite{key1}
		$\tau_3$ & 1 & [1,2] & Equation 3 & \cite{key1}
		$\tau_4$ & 1 & [1,2] & Equation 4 & \cite{key1}
		$\tau_5$ & 1 & [1,2] & Equation 5 & \cite{key1}
		\hline
		$\alpha_1$ & 1 & (0,2] & Equation 1 & \cite{key1}
		$\alpha_2$ & 1 & (0,2] & Equation 2 & \cite{key1}
		$\alpha_3$ & 1 & (0,2] & Equation 3 & \cite{key1}
		$\alpha_4$ & 1 & (0,2] & Equation 4 & \cite{key1}
		$\alpha_5$ & 1 & (0,2] & Equation 5 & \cite{key1}
		\hline
		$\beta_1$ & 1 & (0,2] & Equation 1 & \cite{key1}
		$\beta_2$ & 1 & (0,2] & Equation 2 & \cite{key1}
		$\beta_3$ & 1 & (0,2] & Equation 3 & \cite{key1}
		$\beta_4$ & 1 & (0,2] & Equation 4 & \cite{key1}
		$\beta_5$ & 1 & (0,2] & Equation 5 & \cite{key1}
	\end{tabular}	
\end{table}



\section{Methods}

\subsection{Algorithms}

\section{Results}

\subsection{Tables}

\centering	
\begin{table}[H]\tiny
	\caption{}	
	\begin{tabular}{p{1cm}p{1cm}|p{4cm}|l}
		\hline	
		Model ID & Topic & Description & Results \\
		\hline 
		\hline 
	\end{tabular}
\end{table}

\subsection{Figures}

\begin{figure}[H]
	\centering
	\begin{minipage}[b]{0.5\linewidth}
	%\includegraphics[scale=0.25]{Example_1_Figure_1.png}
	\end{minipage}\hfill
	\begin{minipage}[b]{0.5\linewidth}
	%\includegraphics[scale=0.25]{Example_1_Figure_2.png}
	\end{minipage}\hfill	
	\begin{minipage}[b]{0.5\linewidth}
	%\includegraphics[scale=0.25]{Example_1_Figure_3.png}
	\end{minipage}\hfill
	\begin{minipage}[b]{0.5\linewidth}
	%\includegraphics[scale=0.25]{Example_1_Figure_4.png}
	\end{minipage}\hfill
	\caption{1, 2, 3 and 4}
	\label{fig:Figure1}
\end{figure} 

\subsection{Classroom Discussion}

\begin{enumerate}
\end{enumerate}

\section{Bibliography}

\bibliographystyle{plain}
\begin{thebibliography}{00}

\bibitem[1]{key1}Arwen Meister, Ye Henry Li, Bokyung Choi and Wing Hung Wong
\newblock Learning A Nonlinear Dynamical System Model Of Gene Regulation: A Perturbed Steady-State Approach
\newblock The Annals of Applied Statistics 2013, Vol. 7, No. 3, 1311â€“1333.

\bibitem[2]{key2}H. Frederik Nijhout, Janet A. Best and Michael C. Reed
\newblock Using mathematical models to understand metabolism, genes, and disease
\newblock BMC Biology (2015) 13:79.

\subsection{Differential Equations}

\bibitem[1]{key1}A New Approach and Solution Technique to Solve Time Fractional Nonlinear Reaction-Diffusion Equations
\bibitem[1]{key1}Stability Analysis of Fractional-Order Nonlinear Systems with Delay
\bibitem[1]{key1}Application of the Multistep Generalized Differential Transform Method to Solve a Time-Fractional Enzyme Kinetics
\bibitem[1]{key1}Wavelet Methods for Solving Fractional Order Differential Equations
\bibitem[1]{key1}Numerical Methods for Pricing American Options with Time-Fractional PDE Models
\bibitem[1]{key1}Application of Multistep Generalized Differential Transform Method for the Solutions of the Fractional-Order Chua System
\bibitem[1]{key1}Numerical Solution of Some Types of Fractional Optimal Control Problems
\bibitem[1]{key1}An Efficient Series Solution for Fractional Differential Equations
\bibitem[1]{key1}Approximate Analytical Solution for Nonlinear System of Fractional Differential Equations by BPs Operational Matrices
\bibitem[1]{key1}Numerical Solution for Complex Systems of Fractional Order
\bibitem[1]{key1}Stability Analysis of Fractional-Order Nonlinear Systems with Delay
\bibitem[1]{key1}Numerical Study for Time Delay Multistrain Tuberculosis Model of Fractional Order
\bibitem[1]{key1}A Numerical Method for Solving Fractional Differential Equations by Using Neural Network
\bibitem[1]{key1}Numerical Studies for Fractional-Order Logistic Differential Equation with Two Different Delays
\bibitem[1]{key1}Numerical Modeling of Fractional-Order Biological Systems
\bibitem[1]{key1}Numerical Solution of Some Types of Fractional Optimal Control Problems
\bibitem[1]{key1}A Numerical Method for Delayed Fractional-Order Differential Equations
\bibitem[1]{key1} New Insights into the Fractional Order Diffusion Equation Using Entropy and Kurtosis
\bibitem[1]{key1} DELAY DIFFERENTIAL EQUATIONS IN SINGLE SPECIES DYNAMICS
\bibitem[1]{key1} An Improved Artificial Bee Colony Algorithm Based on Elite Strategy and Dimension Learning
\bibitem[1]{key1}Operators of Fractional Calculus and Their Applications
\bibitem[1]{key1} Modelling Physiological and Pharmacological Control on Cell Proliferation to Optimise Cancer Treatments

\bibitem[400]{key400} Kanehisa, Furumichi, M., Tanabe, M., Sato, Y., and Morishima, K.; 
\newblock KEGG: new perspectives on genomes, pathways, diseases and drugs. 
\newblock Nucleic Acids Res. 45, D353-D361 (2017).

\bibitem[401]{key401} Kanehisa, M., Sato, Y., Kawashima, M., Furumichi, M., and Tanabe, M.; 
\newblock KEGG as a reference resource for gene and protein annotation. 
\newblock Nucleic Acids Res. 44, D457-D462 (2016).

\bibitem[402]{key402} Kanehisa, M. and Goto, S.; 
\newblock KEGG: Kyoto Encyclopedia of Genes and Genomes. 
\newblock Nucleic Acids Res. 28, 27-30 (2000). 

\bibitem[403]{key403} Rouillard AD, Gundersen GW, Fernandez NF, Wang Z, Monteiro CD, McDermott MG, Ma'ayan A. 
\newblock The harmonizome: a collection of processed datasets gathered to serve and mine knowledge about genes and proteins. 
\newblock Database (Oxford). 2016 Jul 3;2016. pii: baw100.

\bibitem[501]{key501} Hadley Wickham, Jim Hester and Romain Francois (2017). readr: Read
\newblock Rectangular Text Data. R package version 1.1.1.
\newblock https://CRAN.R-project.org/package=readr

\bibitem[502]{key502} David B. Dahl (2016). 
\newblock xtable: Export Tables to LaTeX or HTML. R package version 1.8-2.
\newblock https://CRAN.R-project.org/package=xtable

\bibitem[503]{key503}  Feinerer and Kurt Hornik (2017).
\newblock tm: Text Mining Package. 
\newblock R package version 0.7-1. https://CRAN.R-project.org/package=tm

\bibitem[504]{key504}  Feinerer, Kurt Hornik, and David Meyer (2008). 
\newblock Text Mining Infrastructure in R. 
\newblock Journal of Statistical Software 25(5): 1-54. URL:http://www.jstatsoft.org/v25/i05/.

\bibitem[505]{key505} B and Hornik K (2011).
\newblock topicmodels: An R Package for Fitting Topic Models.
\newblock Journal of Statistical Software, *40*(13), pp. 1-30. doi:10.18637/jss.v040.i13 (URL:http://doi.org/10.18637/jss.v040.i13).

\bibitem[506]{key506} Ian Fellows (2014). 
\newblock wordcloud: Word Clouds. R package version 2.5.
\newblock https://CRAN.R-project.org/package=wordcloud

\bibitem[507]{key507} Gagolewski M. and others (2017). 
\newblock R package stringi: Character string processing facilities. 
\newblock http://www.gagolewski.com/software/stringi/.DOI:10.5281/zenodo.32557

\bibitem[508]{key508} Hadley Wickham (2017).
\newblock stringr: Simple, Consistent Wrappers for Common String Operations. 
\newblock R package version 1.2.0. https://CRAN.R-project.org/package=stringr

\bibitem[1000]{key1000}R Core Team (2015). 
\newblock R: A language and environment for statistical computing. R Foundation for Statistical Computing, Vienna, Austria.
\newblock URL https://www.R-project.org/.

\end{thebibliography}

\end{document}
