%-------------------------------------------------------------------------%
%--------------------Classroom Lecture Model Series-----------------------%
%-------------------------------------------------------------------------%
\begin{document}
\twocolumn
\scriptsize
\begin{frontmatter}
		\title{DRAFT Sample: A Mathematical Model of Teleomere Maintenance and Telomeric DNA Binding }
		\author{\corref{cor1}\fnref{fn1}}
		\cortext[cor1]{Corresponding author}
		\address{The Mathematical Learning Space}
		\ead{http://mathlearningspace.weebly.com}	
\end{frontmatter}	

Introduction:
\begin{enumerate}
\item Objective 1:
\item Objective 2:
\item Objective 3:
\end{enumerate}
Conclusion:

Keywords: Signal Transduction, Gene Mutations, Cancer
Vocabulary Words:

\section{Introduction}

\begin{table}[H]\centering
	\begin{tabular}{p{1cm}p{4cm}p{3cm}}
		Article ID & Summary & Comments\\
		\hline
		\hline
	\end{tabular}
\end{table}

\subsection{Biology Review Topic: Signal Transduction Networks}

\begin{table}[H]\centering
	\begin{tabular}{p{1cm}p{4cm}p{3cm}}
		Article ID & Summary & Comments\\
		\hline
		\hline
	\end{tabular}
\end{table}

\subsection{Biology Review Topic: Cancer Mutations}

\begin{table}[H]\centering
	\begin{tabular}{p{1cm}p{4cm}p{3cm}}
		Article ID & Summary & Comments\\
		\hline
		\hline
	\end{tabular}
\end{table}


\begin{table}[H]
\tiny
\caption{Gene Mutations (GM) for Cancer Type (CT) and Signal Transduction Pathway (STP)}
\begin{tabular}{rp{2cm}p{4cm}p{0.5cm}}
\hline 
ID & CT or STP & GM & Comment \\ 
\hline 	
\hline
\end{tabular}
\end{table}

\subsection{Plan of the Article}

\begin{enumerate}
\end{enumerate}


\section{Mathematical Background}

\subsection{Equation System A}

\begin{align*} 
\tiny
\frac{d^{\alpha_1}X_1(t)}{dt^{\alpha_1}} = a_{11} *X_1(t - \tau_1) + \\
a_{12} *\frac{X_1(t)^{\beta_1}}{(1-X_1(t - \tau_1))^{\beta_1}} - a_{15}X_1(t) + \\
\epsilon_1(t) \\
\frac{d^{\alpha_1}X_2(t)}{dt^{\alpha_1}} = a_{21} *X_2(t - \tau_2) + \\
a_{22} *\frac{X_2(t)^{\beta_1}}{(1-X_2(t - \tau_2))^{\beta_1}} - a_{25}X_2(t) + \\
\epsilon_2(t) \\ \\
\frac{d^{\alpha_1}X_3(t)}{dt^{\alpha_1}} = a_{31} *X_3(t - \tau_3) + \\
a_{32} *\frac{X_3(t)^{\beta_1}}{(1-X_3(t - \tau_3))^{\beta_1}} - a_{35}X_3(t) + \\
\epsilon_3(t) \\ \\
\frac{d^{\alpha_1}X_4(t)}{dt^{\alpha_1}} = a_{41} *X_4(t - \tau_4) + \\
a_{42} *\frac{X_4(t)^{\beta_1}}{(1-X_4(t - \tau_4))^{\beta_1}} - a_{45}X_4(t) + \\
\epsilon_4(t) \\ \\
\frac{d^{\alpha_1}X_5(t)}{dt^{\alpha_1}} = a_{51} *X_5(t - \tau_5) + \\
a_{52} *\frac{X_5(t)^{\beta_1}}{(1-X_5(t - \tau_5))^{\beta_1}} - a_{55}X_5(t) + \\
\epsilon_5(t) \\
\end{align*}

\subsection{Parameter Table}

\begin{table}[h]\footnotesize
	\caption{Parameter Description and Value}
	\begin{tabular}{rllp{2cm}l}
		\hline	
		Parameter & Value & Interval & Description & Reference \\
		\hline 
		a11 & 0 & [0,1] & Equation 1 & \cite{key1}
		a12 & 0 & [0,1] & Equation 1 & \cite{key1}
		a13 & 0 & [0,1] & Equation 1 & \cite{key1}
		a14 & 0 & [0,1] & Equation 1 & \cite{key1}
		a15 & 0 & [0,1] & Equation 1 & \cite{key1}
		\hline
		a21 & 0 & [0,1] & Equation 2 & \cite{key1}
		a22 & 0 & [0,1] & Equation 2 & \cite{key1}
		a23 & 0 & [0,1] & Equation 2 & \cite{key1}
		a24 & 0 & [0,1] & Equation 2 & \cite{key1}
		a25 & 0 & [0,1] & Equation 2 & \cite{key1}
		\hline
		a31 & 0 & [0,1] & Equation 3 & \cite{key1}
		a32 & 0 & [0,1] & Equation 3 & \cite{key1}
		a33 & 0 & [0,1] & Equation 3 & \cite{key1}
		a34 & 0 & [0,1] & Equation 3 & \cite{key1}
		a35 & 0 & [0,1] & Equation 3 & \cite{key1}
		\hline
		a41 & 0 & [0,1] & Equation 4 & \cite{key1}
		a42 & 0 & [0,1] & Equation 4 & \cite{key1}
		a43 & 0 & [0,1] & Equation 4 & \cite{key1}
		a44 & 0 & [0,1] & Equation 4 & \cite{key1}
		a45 & 0 & [0,1] & Equation 4 & \cite{key1}
		\hline
		a51 & 0 & [0,1] & Equation 5 & \cite{key1}
		a52 & 0 & [0,1] & Equation 5 & \cite{key1}
		a53 & 0 & [0,1] & Equation 5 & \cite{key1}
		a54 & 0 & [0,1] & Equation 5 & \cite{key1}
		a55 & 0 & [0,1] & Equation 5 & \cite{key1}
		\hline
		$\tau_1$ & 1 & [1,2] & Equation 1 & \cite{key1}
		$\tau_2$ & 1 & [1,2] & Equation 2 & \cite{key1}
		$\tau_3$ & 1 & [1,2] & Equation 3 & \cite{key1}
		$\tau_4$ & 1 & [1,2] & Equation 4 & \cite{key1}
		$\tau_5$ & 1 & [1,2] & Equation 5 & \cite{key1}
		\hline
		$\alpha_1$ & 1 & (0,2] & Equation 1 & \cite{key1}
		$\alpha_2$ & 1 & (0,2] & Equation 2 & \cite{key1}
		$\alpha_3$ & 1 & (0,2] & Equation 3 & \cite{key1}
		$\alpha_4$ & 1 & (0,2] & Equation 4 & \cite{key1}
		$\alpha_5$ & 1 & (0,2] & Equation 5 & \cite{key1}
		\hline
		$\beta_1$ & 1 & (0,2] & Equation 1 & \cite{key1}
		$\beta_2$ & 1 & (0,2] & Equation 2 & \cite{key1}
		$\beta_3$ & 1 & (0,2] & Equation 3 & \cite{key1}
		$\beta_4$ & 1 & (0,2] & Equation 4 & \cite{key1}
		$\beta_5$ & 1 & (0,2] & Equation 5 & \cite{key1}
	\end{tabular}	
\end{table}


\centering
\begin{table}[H]\footnotesize
	\caption{}
	\begin{tabular}{rp{1cm}p{2cm}p{3cm}p{1cm}}
		\hline
		ID & A & B & C & Reference \\
		\hline
		\hline
	\end{tabular}
\end{table}
\raggedright

\subsection{QSAR Properties}

\subsubsection{Molecular Descriptors}

\begin{table}[H]\centering
	\begin{tabular}{p{1cm}p{2cm}p{4cm}p{3cm}}
		ID & Descriptor & Summary & Comments\\
		\hline
		\hline
	\end{tabular}
\end{table}


\begin{figure}[H]
%--------------------------------------------------------------------------------------------
%     Motif A
%--------------------------------------------------------------------------------------------
\begin{tikzpicture}[scale=0.5]
\begin{axis}[width=0.5\textwidth, xlabel={$T-hours$},ylabel={$Y$}]
\addplot[domain=0:48,color=red] {10};
\addplot[domain=0:48,color=blue]{10+cos(2^(4)*x)};
\node[above,red] at (80,0.1) {\footnotesize (A) Y=10};
\node[above,blue] at(150,180) {\footnotesize (B) Y=$10+sin(2^{4}x)$};
\end{axis}
\end{tikzpicture}
%--------------------------------------------------------------------------------------------
%     Motif B
%--------------------------------------------------------------------------------------------
\begin{tikzpicture}[scale=0.5]
\begin{axis}[width=0.5\textwidth, xlabel={$T-hours$},ylabel={$Y$}]
\addplot[domain=0:48,color=red] {5};
\addplot[domain=0:48,color=blue]{5+sin(2^(6)*x)};
\node[above,red] at (80,0.1) {\footnotesize Y=5};
\node[above,blue] at(150,180) {\footnotesize (B) Y=$5+sin(2^{6}x)$};
\end{axis}
\end{tikzpicture}
\caption{}
\end{figure}

\section{Results}


\centering	
\begin{table}[H]\tiny
 \caption{Gene/Protein/Enzymes Description and References}	
\begin{tabular}{r|p{4cm}|llll}
\hline	
Gene/Protein & Description &  Feature 1 & Feature 2 & Feature 3 & Reference \\
\hline 
\hline 
\end{tabular}
\end{table}
\vspace{4pt}

\centering
\begin{table}[H]\footnotesize
	\caption{}
	\begin{tabular}{rp{1cm}p{2cm}p{3cm}p{1cm}}
		\hline
		ID & A & B & C & Reference \\
		\hline
		\hline
	\end{tabular}
\end{table}
\raggedright

\subsection{Figures}

\begin{figure}[H]
	\centering
	\begin{minipage}[b]{0.5\linewidth}
	%\includegraphics[scale=0.25]{Example_1_Figure_1.png}
	\end{minipage}\hfill
	\begin{minipage}[b]{0.5\linewidth}
	%\includegraphics[scale=0.25]{Example_1_Figure_2.png}
	\end{minipage}\hfill	
	\begin{minipage}[b]{0.5\linewidth}
	%\includegraphics[scale=0.25]{Example_1_Figure_3.png}
	\end{minipage}\hfill
	\begin{minipage}[b]{0.5\linewidth}
	%\includegraphics[scale=0.25]{Example_1_Figure_4.png}
	\end{minipage}\hfill
	\caption{1, 2, 3 and 4}
	\label{fig:Figure1}
\end{figure} 

\section{Conclusions}


\section{Topics for the Classroom}

\begin{enumerate}
\end{enumerate}

\section{R Application Programming Interfaces (APIs)}




\bibliographystyle{plain}
\begin{thebibliography}{00}

\bibitem[4]{key4} D.A.J. van Zwieten1, J.E. Rooda1,D.Armbruster and J.D. Nagy
\newblock Simulating feedback and reversibility in substrate-enzyme reactions
\newblock Eur. Phys. J. B 84, 673â€“684 (2011) DOI: 10.1140/epjb/e2011-10911-x

\bibitem[5]{key5} Pilwon Kima and Chang Hyeong Leeb
\newblock A probability generating function method for stochastic reaction networks
\newblock THE JOURNAL OF CHEMICAL PHYSICS 136, 234108 (2012)

\bibitem[400]{key400} Kanehisa, Furumichi, M., Tanabe, M., Sato, Y., and Morishima, K.; 
\newblock KEGG: new perspectives on genomes, pathways, diseases and drugs. 
\newblock Nucleic Acids Res. 45, D353-D361 (2017).

\bibitem[401]{key401} Kanehisa, M., Sato, Y., Kawashima, M., Furumichi, M., and Tanabe, M.; 
\newblock KEGG as a reference resource for gene and protein annotation. 
\newblock Nucleic Acids Res. 44, D457-D462 (2016).

\bibitem[402]{key402} Kanehisa, M. and Goto, S.; 
\newblock KEGG: Kyoto Encyclopedia of Genes and Genomes. 
\newblock Nucleic Acids Res. 28, 27-30 (2000). 

\bibitem[403]{key403} Rouillard AD, Gundersen GW, Fernandez NF, Wang Z, Monteiro CD, McDermott MG, Ma'ayan A. 
\newblock The harmonizome: a collection of processed datasets gathered to serve and mine knowledge about genes and proteins. 
\newblock Database (Oxford). 2016 Jul 3;2016. pii: baw100.

\bibitem[1000]{key1000}R Core Team (2015). 
\newblock R: A language and environment for statistical computing. R Foundation for Statistical Computing, Vienna, Austria.
\newblock URL https://www.R-project.org/.


\end{thebibliography}
\end{document}
