\documentclass[preprint, 8pt]{elsarticle}
\usepackage{xcolor}
\usepackage{chemfig}
\usepackage{tikz}
\usepackage{graphicx}
\usepackage{amsmath, amssymb}
\setlength{\parindent}{0pt}
\usepackage{pgfplots}
\pgfplotsset{compat=1.3,}
\pgfplotscreateplotcyclelist{line styles}{ 
	black,solid\\
	blue,dashed\\
	red,dotted\\
	orange,dashdotted\\
}
\newcommand*\GnuplotDefs{
	set samples 50;
	cdfn(x,mu,sd) = 0.5 * ( 1 + erf( (x-mu)/sd/sqrt(2)) );
	pdfn(x,mu,sd) = 1/(sd*sqrt(2*pi)) * exp( -(x-mu)^2 / (2*sd^2) );
	tpdfn(x,mu,sd,a,b) = pdfn(x,mu,sd) / ( cdfn(b,mu,sd) - cdfn(a,mu,sd) );
}
\usepackage[a4paper,bindingoffset=0.2in,
            left=0.5in,right=0.5in,top=1in,bottom=1in,
            footskip=.25in]{geometry}
%\usepackage{geometry}
\usepackage{mathtools}
\usepackage{tkz-berge}
 \usetikzlibrary{calc} 
\usetikzlibrary{automata}
\usetikzlibrary{arrows}
\usetikzlibrary{positioning,shapes,shadows,arrows}
\usetikzlibrary{shapes.geometric}
\usetikzlibrary{calendar,shadings}
\renewcommand*{\familydefault}{\sfdefault}
\colorlet{winter}{blue}
\colorlet{spring}{green!60!black}
\colorlet{summer}{orange}
\colorlet{fall}{red}
\newcount\mycount
\newcommand\shapeLarge{50mm}
\newcommand\shapeMedium{25mm}
\newcommand\shapeSmall{5mm}
\newcommand*{\xMin}{0}
\newcommand*{\xMax}{6}
\newcommand*{\yMin}{0}
\newcommand*{\yMax}{6}
\newcommand*{\zMax}{6}
\newcommand*{\zMin}{0}
\definecolor{colorwaveA}{RGB}{98,145,224}
\definecolor{colorwaveB}{RGB}{250,250,50}
\definecolor{colorwaveC}{RGB}{25,125,25}
\definecolor{colorwaveD}{RGB}{100,100,100}
\definecolor{colorwaveE}{RGB}{80,100,1}
\definecolor{colorwaveF}{RGB}{60,1,1}
\definecolor{colorwaveG}{RGB}{25,1,100}
\definecolor{colorwaveH}{RGB}{1,90,1}
\definecolor{colorwaveI}{RGB}{1,100,1}
\definecolor{colorwaveJ}{RGB}{1,1,1}
\tikzset{
	shapeTriangle/.style={draw,shape=regular polygon,fill=colorwaveA,circular drop shadow,regular polygon sides=3,minimum size=\shapeSmall,inner sep=0pt,outer sep=0pt},
	shapeTriangle3/.style={shapeTriangle,fill=colorwaveD,circular drop shadow,shape border rotate=45},
	shapeTriangle4/.style={shapeTriangle,fill=colorwaveA,circular drop shadow,shape border rotate=90},
	shapeTriangle5/.style={shapeTriangle,fill=colorwaveB,shape border rotate=135},
	shapeTriangle6/.style={shapeTriangle,fill=colorwaveC,shape border rotate=180},
	shapeTriangle7/.style={shapeTriangle,fill=colorwaveE,shape border rotate=225},
	shapeTriangle8/.style={shapeTriangle,fill=colorwaveF,shape border rotate=270},
	shapeTriangle9/.style={shapeTriangle,fill=colorwaveG,shape border rotate=315},
}
\tikzset{shapeUgaritic/.style={draw,shape=regular polygon,fill=colorwaveD,circular drop shadow,regular polygon sides=3,minimum size=\shapeSmall,inner sep=0pt,outer sep=0pt},
}

\tikzset{
	shapeSquare/.style={draw,shape=regular polygon,fill=colorwaveC,circular drop shadow,regular polygon sides=4,minimum size=\shapeSmall,inner sep=0pt,outer sep=0pt},
	shapeSquare2/.style={shapeSquare,shape border rotate=45},
}

\tikzset{
	shapeHexagon/.style={draw,shape=regular polygon,fill=colorwaveA,circular drop shadow,regular polygon sides=6,minimum size=\shapeSmall,inner sep=0pt,outer sep=0pt},
	shapeHexagon2/.style={shapeHexagon,shape border rotate=90},
}

\tikzset{
	shapeOctagon/.style={draw,shape=regular polygon,fill=colorwaveB,circular drop shadow,regular polygon sides=8,minimum size=\shapeSmall,inner sep=0pt,outer sep=0pt},
	shapeOctagon2/.style={shapeHexagon,shape border rotate=45},
}
\tikzset{
	shapeEllipse/.style={draw,shape=ellipse,minimum size=\shapeSmall,inner sep=0pt,outer sep=0pt},
	shapeEllipse2/.style={shapeEllipse,shape border rotate=90},
}

\tikzset{
	closedFigure/.style={draw=\draw[->,rounded corners=0.2cm,shorten >=2pt]
		(1.5,0.5)-- ++(0,-1)-- ++(1,0)-- ++(0,2)-- ++(-1,0)-- ++(0,2)-- ++(1,0)--
		++(0,1)-- ++(-1,0)-- ++(0,-1)-- ++(-2,0)-- ++(0,3)-- ++(2,0)-- ++(0,-1)--
		++(1,0)-- ++(0,1)-- ++(1,0)-- ++(0,-1)-- ++(1,0)-- ++(0,-3)-- ++(-2,0)--
		++(1,0)-- ++(0,-3)-- ++(1,0)-- ++(0,-1)-- ++(-6,0)-- ++(0,3)-- ++(2,0)--
		++(0,-1)-- ++(1,0)}
}
\tikzstyle{start}=[circle, draw=none,,minimum size=\shapeMedium, fill=blue, circular drop shadow,text centered, anchor=north, text=white]
\tikzstyle{finish}=[circle, draw=none,,minimum size=\shapeMedium, fill=blue,circular drop shadow,text centered, anchor=north, text=white]
\tikzstyle{finish}=[rectangle, draw=none, ,minimum size=\shapeMedium,fill=blue,circular drop shadow,text centered, anchor=north, text=white]
\usepackage[noadjust]{cite}
\usepackage{algpseudocode}
\usepackage{listings}
\usepackage{algorithm}
\usepackage{color}
\usepackage{parskip}
\usepackage{amsfonts}
\usepackage{amsthm}
\usepackage{tikz}
\usepackage{tkz-berge}
\usepackage{caption}
\usepackage{hyperref}
\usepackage{amsrefs}
\usepackage{mathtools, amssymb}
\usepackage{graphicx}
\usepackage{subcaption}
\usepackage{tabularx,ragged2e}
\usepackage[framemethod=tikz]{mdframed}
\newcommand{\N}{\mathbb N}
\newcommand{\Q}{\mathbb Q}
\theoremstyle{definition}
\newtheorem{definition}{Definition}[section] 
\newtheorem{theorem}{Theorem}[section]
\newtheorem{example}{Example}[section]
\renewcommand{\qedsymbol}{$\blacksquare$}
\newtheorem{corollary}{Corollary}[theorem]
\newtheorem{lemma}[theorem]{Lemma}
\renewcommand{\rmdefault}{ptm} 
\graphicspath{{Figures/}}
\twocolumn

\begin{document}
	
keywords: 35A24 Methods of ordinary differential equations, 37C10 Vector Fields, flows and ordinary differential equations,37H10 Generation, random and stochastic difference and differential equations, 11K70 Harmonic analysis and almost periodicity

\section{Introduction}

\begin{enumerate}
\end{enumerate}

\section{Material and Methods}

\section{Networks}

A sample of one arrangment of four separate networks for the dynamical description of (A) DNA damage repair (B) Cell Cycle Arrest  (C) Oncogene dynamics with PI3K /PTEN/Akt signaling pathway, and (D) Apoptosome Formation for Apoptosis. Figure 1 shows the relationships. 

\begin{figure}[H]
\begin{tikzpicture}[auto, thick, node distance=1.25cm, >=triangle 45,scale=.15]
\tiny
\draw
	%--------------------------------------------------Drawing of Network A---------------------------------------------%
	node at (0,0)[right=-3mm]{\small Start}
	node [input, name=input1] {} 
	node [block, right of=input1] (UV) {UV}
	node [block, below of=UV] (ATR) {ATR}
        node [block, right of=UV] (DNA) {DNA Damage}
	node[block, left of=ATR](RPA) {RPA}
        node [block, below of=DNA] (ATRp) {ATRp};
  	
	\draw[->] (input1) --(UV);   
	\draw[->] (UV) --(DNA); 
	\draw[->] (ATR) --(ATRp); 
	\draw[->,dashed] (RPA) --(ATR); 
	\draw[->,dashed] (DNA)--(12,-8);
	%\draw [color=gray,thick](0,0) rectangle (1,1);
	\node at (1,3) [above=1mm, right=0mm] {\textsc{Network A}};	
	%------------------------------------------------Drawing Network B------------------------------------------------%
	\draw
	node at (30,1) [block] (p21) {p21}
    	node [block, right of=p21] (DDb2) {DDb2}
    	node [block, below of=DDb2] (Rbp) {Rbp}
    	node [block, below of=Rbp] (Cyce) {Cyce}
	node [block, right of=Rbp] (RB) {RB}
	node [block, below of=RB] (EF21) {EF21}
	node at (30,-5) [block](CC) {Cell Cycle}
	;
	\draw[->] (p21) --(CC);  
	 \draw[->] (RB) --(EF21);
	 \draw[->] (RB) --(Rbp);
	 \draw[->] (Rbp) --(RB);
	 \draw[->] (Rbp) --(EF21);
	 \draw[->] (DDb2) --(p21);
	 \draw[->] (EF21) --(Cyce);
	 \draw[->] (p21) --(Cyce);
	%\draw [color=gray,thick](5.5,-3) rectangle (12,2);
	\node at (35,4) [below=5mm, right=0mm] {\textsc{Network B}};
	%----------------------------------------------- Draw Network C-------------------------------------------------%
	\draw
	node at  (0.5,-20) [sum, name=p53] {P53}
	node [sum, right of=p53] (mdm2n) {Mdm2n}
	node [block, right of=mdm2n] (mdm2cp) {Mdm2cp}
	node [block, right of=mdm2cp] (aktp) {Aktp}
	node [block, below of=p53] (p53p) {p53p}
    	node [block, right of=p53p] (pten) {Pten}
	node [block, below of=mdm2cp] (mdm2c) {Mdm2c}
	node [block, right of=mdm2c] (akt) {Akt}
	node [block, below of=pten] (pip2) {PIP2}
	node [block, below of=mdm2c] (pip3) {PIP3}
	;
	\draw[->] (p53) --(p53p);  
	\draw[->,dashed] (p53p) --(p53); 
	\draw[->] (ATRp) --(mdm2n);  
	\draw[->,dashed] (p21) --(p53);  
	 \draw[->] (pip2) --(pip3);
 	\draw[->] (pip3) --(pip2);
 	\draw[->] (akt) --(aktp);
	 \draw[->] (aktp) --(akt);
	\draw[->] (p53p) --(pten);  
	\draw[->] (mdm2cp) --(mdm2c);  
	\draw[->] (mdm2c) --(mdm2cp);
	\draw[->] (pip3) --(akt);
	\draw[->,dashed] (p53p) --(mdm2c);
	\draw[->] (mdm2n) --(p53);
	%\draw [color=gray,thick](-0.5,-9) rectangle (7,-4);
	\node at (-0.5,-40) [below=5mm, right=0mm] {\textsc{Network C}};
	%----------------------------------------------Draw Network D-----------------------------------------------%
	\draw
	node  at (30, -20) [block] (bax) {BAX}
	node [block,below of=bax ](cytoc) {CytoC}
	node [block, right of=bax] (apops) {Apops}
	node [block, below of=apops] (casp9) {CASP9}
	node [block, right of=apops] (procasp3) {Procasp3}
	node [block, below of=cytoc] (apaf1) {Apaf-1}
	node [block, below  of=casp9] (procasp9) {Procasp9}
	node [block, below of=procasp3] (casp3) {casp3}
	node [block, below of=apaf1] (APTX) {APTX}
	node [block, below of=casp3] (PARP1) {PARP1}
	node [block, below of=PARP1] (PARP3) {PARP3}
	node[sum, below of=PARP3] (Apoptosis){Apoptosis}
	;
	\draw[->,dashed] (p53) to [controls=+(30:12) and +(30:12)] (bax);
	\draw[->,dashed] (bax) --(cytoc);
  	\draw[->,dashed] (cytoc) --(apops);
	\draw[->,dashed] (apaf1) --(apops);
	\draw[->,dashed] (apops) --(casp9);
	\draw[->,dashed] (casp9) --(procasp9);
	\draw[->,dashed] (procasp9) --(casp9);
	\draw[->,dashed] (casp3) --(procasp3);
	\draw[->,dashed] (procasp3) --(casp3);
	\draw[->,dashed] (casp3) --(casp9);
	\draw[->] (PARP1) --(APTX); 
	\draw[->] (PARP1) --(PARP3); 
	\draw[->] (casp3) --(PARP1); 
	\draw[->] (bax) --(casp3); 
	\draw[->] (PARP3) --(Apoptosis);
	%\draw [color=gray,thick](17.5,-9) rectangle (12.5,-4);
	\node at (35,-60) [below=5mm, right=0mm] {\textsc{Network D}};
\end{tikzpicture}
\end{figure}

\section{Definitions}

\section{Theorems}

\section{Algorithms}

\section{Equation Systems}

%--------------------------------------------------P53---------------------------------------------------------------------%
The relationships between p53 and Mdm2 in following set of equations with parameter values in Table.  
\centering
\small
\vspace{4pt}
	\begin{equation}
	\end{equation}
	\begin{equation}
	\begin{split}
	\end{split}
	\end{equation}
	\begin{equation}
	\begin{split}
	\end{split}
	\end{equation} 
	\vspace{4pt}
\raggedright
where $\eta_{6}(t) and \eta{7}(t) \approx$ independently and identically distributed UNIFORM (a,b) where a<b. 
\centering
\begin{table}[H]\footnotesize
\caption{Parameter Description and Value}
\begin{tabular}{rlllp{1.25cm}l}
\hline	
Parameter & Value & Internal & 10 & Description & Reference \\
\hline 
\hline
\end{tabular}


\section{Parameter Table}

Table 1 shows four different sets of values for the parameter vector or matrix.

\vspace{4pt}
\begin{table}[H]\tiny
\caption{Variable Selected Parameter Ranges and Values}
\begin{tabular}{r|>{\tiny}p{1cm}|llll}
		\hline	
		Gene/Protein/Enzyme & Value1 &  Value2 & Value3 & Value4 & Reference \\
		  \hline 
		\hline
\end{tabular}
\end{table}
\vspace{4pt}
\raggedright

\section{Results}

\begin{enumerate}
\end{enumerate}

\vspace{6pt}
\begin{figure}[H]
	\centering
	\begin{minipage}[b]{0.5\linewidth}
%		\includegraphics[scale=0.2]{Figure1.png}
	   \captionof{figure}{Network A}
            \label{fig:networkA}
	\end{minipage}\hfill
	\begin{minipage}[b]{0.5\linewidth}
%		\includegraphics[scale=0.2]{Figure2.png}
 		\captionof{figure}{Network B}
            \label{fig:networkB}
	\end{minipage}\hfill
	\begin{minipage}[b]{0.5\linewidth}
%		\includegraphics[scale=0.2]{Figure3.png}
 	\captionof{figure}{Network C}
            \label{fig:networkC}
	\end{minipage}\hfill
	\begin{minipage}[b]{0.5\linewidth}
%		\includegraphics[scale=0.2]{Figure4.png}
	 \captionof{figure}{Network D}
            \label{fig:networkD}
	\end{minipage}\hfill
	\caption{Model (A), Model (B), Model (C), Model (D) for Different Values of UV}
	\label{fig:Figure1}
\end{figure}

%---------------------------------------------------Network A------------------------------------%

\subsection{Network A: DNA repair with Nucleotide Excision Repair}
\raggedright
Figure 1 shows the relationship between the UV level and response of DNA repair with NER for a single strand break.  Relationship summarized by:

\begin{enumerate}
\end{enumerate}

\vspace{6pt}
\begin{figure}[H]
	\centering
%		\includegraphics[scale=0.35]{Figure1.png}
	\caption{Network A}
	\label{fig:Figure1}
\end{figure}

Table 1 shows the first six moments for the quantities of UV, ATR, ATRp and RPA from the solution of the equation system in the time interval [0,T].
\centering
\begin{table}[H]\tiny
 \caption{Moment Analysis for Network A}
\begin{tabular}{rp{.3cm}p{.3cm}p{.25cm}p{.25cm}p{.25cm}p{.25cm}p{.25cm}p{.25cm}p{.25cm}}
	\hline
	Variable &Min & Max & Moment 1 & Moment 2 & Moment 3 & Moment 4 & Moment 5 & Moment 6 &  $\theta$\\ 
	\hline
	\hline
	\end{tabular}
\end{table}
\raggedright

Table 2 provides the timing and duration for each of the variables in a particular state A: Increase, B: Decrease.  

\begin{enumerate}
\end{enumerate}

\begin{table}[H]\tiny
  \caption{Timing and Duration Model of Network Subunits}
\begin{tabular}{rllp{1.25cm}lll}
		\hline	
		Parameter & A & B & Duration & Entry Time &  Exit Time & Comment \\
		 \hline 
		\hline
\end{tabular}
\end{table}


\section{Discussion Topics for the Classroom}

\begin{enumerate}
\end{enumerate}

\section{Acknowledgements}


\section{Conflict of Interest}

\section{References}

\bibliographystyle{plain}
\begin{thebibliography}{00}
\footnotesize
\bibitem[1]{key1}Li H1, Zhang XP, Liu F. (2013). 
\newblock Coordination between p21 and DDB2 in the cellular response to UV radiation 
\newblock PLoS One. https://www.ncbi.nlm.nih.gov/pubmed/24260342

\bibitem[2]{key20}Fitch ME, Nakajima S, Yasui A, Ford JM (2003) 
\newblock In vivo recruitment of XPC to UV-induced cyclobutane pyrimidine dimers by the DDB2 gene product. 
\newblock J Biol Chem 278: 46906–46910.

\bibitem[3]{key28} Liu S, Shiotani B, Lahiri M, Marechal A, Tse A, et al. (2011) 
\newblock ATR autophosphorylation as a molecular switch for checkpoint activation. 
\newblock Mol Cell 43: 192–202.

\bibitem[4]{key23} Stoyanova T, Roy N, Kopanja D, Bagchi S, Raychaudhuri P (2009) 
\newblock DDB2 decides cell fate following DNA damage. 
\newblock Proc Natl Acad Sci USA 106: 10690–10695.

\bibitem[5]{key11} Abbas T, Dutta A (2009) 
\newblock p21 in cancer: intricate networks and multiple activities. 
\newblock Nat Rev Cancer 9: 400–414.

\bibitem[6]{key34} Mayo LD, Donner DB (2002) 
\newblock The PTEN, Mdm2, p53 tumor suppressor-oncoprotein network. 
\newblock Trends Biochem Sci 27: 462–467.

\bibitem[7]{key39} 39. Song MS, Salmena L, Pandolfi PP (2012) 
\newblock The functions and regulation of the PTEN tumour suppressor. 
\newblock Nat Rev Mol Cell Biol 13: 283–296.

\bibitem[8]{key40} McCubrey JA, Steelman LS, Kempf CR, Chappell WH, Abrams SL, et al. (2011) 
\newblock Therapeutic resistance resulting from mutations in Raf/MEK/ERK and PI3K/PTEN/Akt/mTOR signaling pathways. 
\newblock J Cell Physiol 226: 2762–2781.

\bibitem[9]{key33}. Wu X, Bayle JH, Olson D, Levine AJ (1993) 
\newblock The p53-mdm-2 autoregulatory feedback loop. 
\newblock Genes Dev 7: 1126–1132.

\bibitem[10]{key35}. Stambolic V, MacPherson D, Sas D, Lin Y, Snow B, et al. (2001) 
\newblock Regulation of PTEN transcription by p53. 
\newblock Mol Cell 8: 317–325.

\bibitem[11]{key36}. Stambolic V, Suzuki A, de la Pompa JL, Brothers GM, Mirtsos C, et al. (1998)
\newblock Negative regulation of PKB/Akt-dependent cell survival by the tumor suppressor 
\newblock PTEN. Cell 95: 29–39.

\bibitem[12]{key52}Caldecott KW. 
\newblock Single-strand break repair and genetic disease. 
\newblock Nat Rev Genet. 2008; 9:619–631.[PubMed: 18626472]

\bibitem[13]{key53}Amé JC, Rolli V, Schreiber V, et al. 
\newblock PARP-2, A novel mammalian DNA damage-dependent poly(ADP-ribose) polymerase. 
\newblock J Biol Chem. 1999; 274:17860–17868. [PubMed: 10364231]

\bibitem[14]{key54}Ménissier de Murcia J, Ricoul M, Tartier L, et al. 
\newblock Functional interaction between PARP-1 and PARP-2 in chromosome stability and embryonic development in mouse. 
\newblock EMBOJ. 2003; 22:2255–2263.Tomkinson et al.

\bibitem[15]{key55} Lin, Y., Bai, L., Cupello, S., Hossain, M. A., Deem, B., McLeod, M., … Yan, S. (2018). 
\newblock APE2 promotes DNA damage response pathway from a single-strand break.
\newblock Nucleic Acids Research, 46(5), 2479–2494. http://doi.org/10.1093/nar/gky020

\bibitem[16]{key56} Falck, J., Mailand, N., Syljuasen, R. G., Bartek, J., Lukas, J. 
\newblock The ATM-Chk2-Cdc25A checkpoint pathway guards against radioresistant DNA synthesis. 
\newblock Nature 410: 842-847, 2001. 

\bibitem[17]{key57}Martin, G. A., Bollag, G., McCormick, F., Abo, A. 
\newblock A novel serine kinase activated by rac1/CDC42Hs-dependent autophosphorylation is related to PAK65 and STE20. 
\newblock EMBO J. 14: 1970-1978, 1995. Note: Erratum: EMBO J. 14: 4385 only, 1995. 

\bibitem[18]{key58}Jiang, X., Kim, H.-E., Shu, H., Zhao, Y., Zhang, H., Kofron, J., Donnelly, J., Burns, D., Ng, S., Rosenberg, S., Wang, X. 
\newblock Distinctive roles of PHAP proteins and prothymosin-alpha in a death regulatory pathway. 
\newblock Science 299: 223-226, 2003.

\bibitem[19]{key59}Hollander MC, Blumenthal GM, Dennis PA. 
\newblock PTEN loss in the continuum of common cancers, rare syndromes and mouse models.
\newblock Journal Nat Rev Cancer 11:289-301 (2011)

\bibitem[19]{key60}Li, D.-M., Sun, H. 
\newblock PTEN/MMAC1/TEP1 suppresses the tumorigenicity and induces G1 cell cycle arrest in human glioblastoma cells. 
\newblock Proc. Nat. Acad. Sci. 95: 15406-15411, 1998. 

\bibitem[20]{key61} KH, Sobol RW.
\newblock A unified view of base excision repair: lesion-dependent protein complexes regulated by post-translational modification.
\newblock DNA Repair (Amst) 6:695-711 (2007) DOI:10.1016/j.dnarep.2007.01.009 PMID:17337257

\bibitem[21]{key62} Rouillard AD, Gundersen GW, Fernandez NF, Wang Z, Monteiro CD, McDermott MG, Ma'ayan A. 
\newblock The harmonizome: a collection of processed datasets gathered to serve and mine knowledge about genes and proteins. 
\newblock Database (Oxford). 2016 Jul 3;2016. pii: baw100. 

\subsection{R Packages}

\bibitem[100]{key1000}R Core Team (2015). 
\newblock R: A language and environment for statistical computing. R Foundation for Statistical Computing, Vienna, Austria.
\newblock URL https://www.R-project.org/.

\bibitem[101]{key1001} Hadley Wickham, Jim Hester and Romain Francois (2017). readr: Read
\newblock Rectangular Text Data. R package version 1.1.1.
\newblock https://CRAN.R-project.org/package=readr

\bibitem[102]{key1002} David B. Dahl (2016). 
\newblock xtable: Export Tables to LaTeX or HTML. R package version 1.8-2.
\newblock https://CRAN.R-project.org/package=xtable

\bibitem[103]{key1003}  Feinerer and Kurt Hornik (2017).
\newblock tm: Text Mining Package. 
\newblock R package version 0.7-1. https://CRAN.R-project.org/package=tm

\bibitem[104]{key1004}  Feinerer, Kurt Hornik, and David Meyer (2008). 
\newblock Text Mining Infrastructure in R. 
\newblock Journal of Statistical Software 25(5): 1-54. URL:http://www.jstatsoft.org/v25/i05/.

\bibitem[105]{key1005} B and Hornik K (2011).
\newblock topicmodels: An R Package for Fitting Topic Models.
\newblock Journal of Statistical Software, *40*(13), pp. 1-30. doi:10.18637/jss.v040.i13 (URL:http://doi.org/10.18637/jss.v040.i13).

\bibitem[106]{key1006} Ian Fellows (2014). 
\newblock wordcloud: Word Clouds. R package version 2.5.
\newblock https://CRAN.R-project.org/package=wordcloud

\bibitem[107]{key1007} Gagolewski M. and others (2017). 
\newblock R package stringi: Character string processing facilities. 
\newblock http://www.gagolewski.com/software/stringi/.DOI:10.5281/zenodo.32557

\bibitem[108]{key1008} Hadley Wickham (2017).
\newblock stringr: Simple, Consistent Wrappers for Common String Operations. 
\newblock R package version 1.2.0. https://CRAN.R-project.org/package=stringr

\subsubsection{Scientific Visualization}

\bibitem[201]{key2001}Michael J. Grayling (2014). 
\newblock phaseR:Phase Plane Analysis of One and Two Dimensional Autonomous ODE Systems. 
\newblock R package version 1.3. https://CRAN.R-project.org/package=phaseR

\bibitem[202]{key2002} Wei and Viliam Simko (2016).
\newblock corrplot: Visualization of a Correlation Matrix. 
\newblock R package version 0.77. https://CRAN.R-project.org/package=corrplot

\bibitem[203]{key2003} Soetaert (2017). 
\newblock plot3D: Plotting Multi-Dimensional Data. 
\newblock R package version 1.1.1. https://CRAN.R-project.org/package=plot3D

\bibitem[204]{key2004} Ligges, U. and Mächler, M. (2003).
\newblock Scatterplot3d - an R Package for Visualizing Multivariate Data.
\newblock Journal of Statistical Software 8(11), 1-20.

\bibitem[205]{key2005}Daniel Adler, Duncan Murdoch and others (2017). 
\newblock rgl: 3D Visualization Using OpenGL. 
\newblock R package version 0.98.1. https://CRAN.R-project.org/package=rgl

\bibitem[206]{key2006} Almende B.V., Benoit Thieurmel and Titouan Robert (2017). visNetwork:
\newblock Network Visualization using 'vis.js' Library. R package version 2.0.1.
\newblock https://CRAN.R-project.org/package=visNetwork

\bibitem[207]{key2007} Nicholas Hamilton (2017). 
\newblock ggtern: An Extension to 'ggplot2', for the Creation of Ternary Diagrams. 
\newblock R package version 2.2.1. https://CRAN.R-project.org/package=ggtern

\subsubsection{Mathematics and Statistics}

\bibitem[301]{key3001} Karline Soetaert, Jeff Cash and Francesca Mazzia (2016). 
\newblock deTestSet: Testset for Differential Equations. R package version 1.1.3.
\newblock http://CRAN.R-project.org/package=deTestSet

\bibitem[302]{key3002}Karline Soetaert, Thomas Petzoldt, R. Woodrow Setzer (2010). 
\newblock Solving Differential Equations in R: Package deSolve. 
\newblock Journal of Statistical Software, 33(9), 1--25. URL http://www.jstatsoft.org/v33/i09/ DOI 10.18637/jss.v033.i09

\bibitem[303]{key3003}Soetaert, Karline and Meysman, Filip, (2012).
\newblock Reactive transport in aquatic ecosystems: Rapid model prototyping in the open source software R
\newblock Environmental Modelling and Software, 32, 49-60.

\bibitem[304]{key3004}Soetaert K. and P.M.J. Herman (2009).  
\newblock A Practical Guide to Ecological Modelling. 
\newblock Using R as a Simulation Platform.  Springer, 372 pp.

\bibitem[305]{key3005}Soetaert K. (2009).  
\newblock rootSolve: Nonlinear root finding, equilibrium and steady-state analysis of ordinary differential equations.  
\newblock R-package version 1.6.

\bibitem[306]{key3006}Martin Becker and Stefan Klößner (2016). 
\newblock PearsonDS: Pearson Distribution System. 
\newblock R package version 0.98. http://CRAN.R-project.org/package=PearsonDS

\bibitem[307]{key3007}Csardi G, Nepusz T (2006).
\newblock The igraph software package for complex network research,
\newblock  InterJournal, Complex Systems 1695. 2006. http://igraph.org

\bibitem[308]{key3008} J. O. Ramsay, Hadley Wickham, Spencer Graves and Giles Hooker (2017). 
\newblock fda:Functional Data Analysis. 
\newblock R package version 2.4.7. https://CRAN.R-project.org/package=fda

\bibitem[309]{key3009} Antonio and Fabio Di Narzo (2013).
\newblock tseriesChaos: Analysis of nonlinear time series. 
\newblock R package version 0.1-13.https://CRAN.R-project.org/package=tseriesChaos

\bibitem[310]{key3010} Angelo Canty and Brian Ripley (2017). 
\newblock boot: Bootstrap R (S-Plus) Functions. 
\newblock R package version 1.3-20.

\bibitem[311]{key3011}Yves Tillé and Alina Matei (2016). 
\newblock sampling: Survey Sampling. 
\newblock R package version 2.8. https://CRAN.R-project.org/package=sampling

\bibitem[312]{key3012} Davison, A. C. and Hinkley, D. V. (1997) 
\newblock Bootstrap Methods and Their Applications. 
\newblock Cambridge University Press, Cambridge. ISBN 0-521-57391-2.

\bibitem[313]{key3013}Hans Werner Borchers (2017). 
\newblock pracma: Practical Numerical Math Functions. 
\newblock R package version 2.0.7. https://CRAN.R-project.org/package=pracma

\bibitem[314]{key3014}Marie Laure Delignette-Muller, Christophe Dutang (2015). 
\newblock fitdistrplus:An R Package for Fitting Distributions. 
\newblock Journal of Statistical Software, 64(4), 1-34. URL http://www.jstatsoft.org/v64/i04/.

\bibitem[315]{key3015} Scott Chasalow (2012). 
\newblock combinat: combinatorics utilities. 
\newblock R package version 0.0-8. https://CRAN.R-project.org/package=combinat

\bibitem[316]{key3016} C. Dutang, V. Goulet and M. Pigeon (2008). 
\newblock actuar: An R Package for Actuarial Science. 
\newblock Journal of Statistical Software, vol. 25, no. 7, 1-37. URL http://www.jstatsoft.org/v25/i07

\bibitem[317]{key3017} Douglas Bates and Martin Maechler (2017). 
\newblock Matrix: Sparse and Dense Matrix Classes and Methods. 
\newblock R package version 1.2-11.https://CRAN.R-project.org/package=Matrix

\bibitem[318]{key3018} Killick R, Haynes K and Eckley IA (2016).
\newblock changepoint: An R package for changepoint analysis. R package version
\newblock 2.2.2, <URL: https://CRAN.R-project.org/package=changepoint>.

\subsection{Genomics}

\bibitem[400]{key400} Kanehisa, Furumichi, M., Tanabe, M., Sato, Y., and Morishima, K.; 
\newblock KEGG: new perspectives on genomes, pathways, diseases and drugs. 
\newblock Nucleic Acids Res. 45, D353-D361 (2017).

\bibitem[401]{key401} Kanehisa, M., Sato, Y., Kawashima, M., Furumichi, M., and Tanabe, M.; 
\newblock KEGG as a reference resource for gene and protein annotation. 
\newblock Nucleic Acids Res. 44, D457-D462 (2016).

\bibitem[402]{key402} Kanehisa, M. and Goto, S.; 
\newblock KEGG: Kyoto Encyclopedia of Genes and Genomes. 
\newblock Nucleic Acids Res. 28, 27-30 (2000). 

\bibitem[403]{key403} Rouillard AD, Gundersen GW, Fernandez NF, Wang Z, Monteiro CD, McDermott MG, Ma'ayan A. 
\newblock The harmonizome: a collection of processed datasets gathered to serve and mine knowledge about genes and proteins. 
\newblock Database (Oxford). 2016 Jul 3;2016. pii: baw100. 

\bibitem[500]{key500} J. Timmer and M. König (1995): 
\newblock On generating power law noise. 
\newblock Astron. Astrophys. 300, 707-710.

\subsection{Bioconductor:Genomics}

\bibitem[4001]{key4001}D. Charif and J.R. Lobry (2007)
\newblock Structural approaches to sequence evolution: Molecules, networks, populations edited U. Bastolla and M. Porto and H.E. Roman and M. Vendruscolo 
\newblock New York: Springer Verlag

\bibitem[4002]{key4002} Jitao David Zhang and Stefan Wiemann (2009)
\newblock \emph{KEGGgraph: a graph approach to KEGG PATHWAY in R and Bioconductor}.
\newblock Bioinformatics, 25(11):1470--1471

\bibitem[4003]{key4003} Jitao David Zhang (2017). 
\newblock KEGGgraph: Application ExamplesR
\newblock package version 1.38.1.

\bibitem[4004]{key4004} David Winter (2017). 
\newblock rentrez: Entrez in R. 
\newblock R package version 1.1.0. https://CRAN.R-project.org/package=rentrez

\bibitem[4005]{key4005} H. Pagès, P. Aboyoun, R. Gentleman and S. DebRoy (2017).
\newblock Biostrings: String objects representing biological sequences, and matching algorithms. 
\newblock R package version 2.44.2.

\bibitem[4006]{key4006} Vesna Memisevic (2016). 
\newblock sysBio: A package for modeling biological systems. 
\newblock R package version 1.0.0. https://github.com/Vessy/sysBio

\bibitem[4007]{key4007} Philipp H Boersch-Supan, Sadie J Ryan, and Leah R Johnson (2016). 
\newblock deBInfer: Bayesian inference for dynamical models of biological systems in R. 
\newblock arXiv:1605.00021. URL https://arxiv.org/abs/1605.00021


\subsection{R References}

\bibitem[1000]{key1000}R Core Team (2015). 
\newblock R: A language and environment for statistical computing. R Foundation for Statistical Computing, Vienna, Austria.
\newblock URL https://www.R-project.org/.

\end{thebibliography}

\end{document}
